import os
from pathlib import Path


structure_module_c_s = 384
structure_module_c_z = 128
c_s = 1024
c_z = 128

ICLR_MODEL_ID = "5j007z42"


# DEFAULT_CACHE = Path(os.environ['HOME']) / f".cache/cheap"
DEFAULT_CACHE = Path("/data/lux70/cheap")


# Weights to trained latent-to-sequence decoder
DECODER_CKPT_PATH = Path(DEFAULT_CACHE) / "sequence_decoder/mlp.ckpt"


# Directory to where per-channel statistics are stored
TENSOR_STATS_DIR = Path(DEFAULT_CACHE) / "statistics"


# Directory to where pre-trained models are stored
CHECKPOINT_DIR_PATH = Path(DEFAULT_CACHE) / "checkpoints"


ACCEPTED_LM_EMBEDDER_TYPES = [
    "esmfold",  # 1024 -- i.e. t36_3B with projection layers, used for final model
    "esmfold_pre_mlp",  # 2560
    "esm2_t48_15B_UR50D",  # 5120
    "esm2_t36_3B_UR50D",  # 2560
    "esm2_t33_650M_UR50D",  # 1280
    "esm2_t30_150M_UR50D",  # 64e $EMBED
    "esm2_t12_35M_UR50D",  # dim=480
    "esm2_t6_8M_UR50D",  # dim=320
]

COMPRESSED_DATA_STDS = {
    "qjs33lme": 0.3685586,
    "jzlv54wl": 0.5509415,
    "wiepwn5p": 0.5558105,
    "h9hzw1bp": 0.2876465,
    "j1v1wv6w": 0.3381734,
}


COMPRESSION_INPUT_DIMENSIONS = {
    "qjs33lme": 8,
    "jzlv54wl": 8,
    "wiepwn5p": 8,
    "h9hzw1bp": 64,
    "j1v1wv6w": 32,
    ## CATH, len 256 ##
    "8ebs7j9h": 4,
    "mm9fe6x9": 8,
    "kyytc8i9": 16,
    "fbbrfqzk": 32,
    "13lltqha": 64,
    "uhg29zk4": 128,
    "ich20c3q": 256,
    "7str7fhl": 512,
    "g8e83omk": 1024,
    ## CATH, len 512 ##
    "1b64t79h": 4,
    "1hr1x9r5": 8,
    "yfel5fnl": 16,
    "v2cer77t": 32,
    "2tjrgcde": 64,
    "3rs1hxky": 128,
    "5z4iaak9": 256,
    "q3m9fhii": 512,
}


COMPRESSION_SHORTEN_FACTORS = {
    "qjs33lme": 2,
    "jzlv54wl": 1,
    "wiepwn5p": 1,
    "h9hzw1bp": 2,
    "j1v1wv6w": 2,
    ## CATH, len 256 ##
    "8ebs7j9h": 2,
    "mm9fe6x9": 2,
    "kyytc8i9": 2,
    "fbbrfqzk": 2,
    "13lltqha": 2,
    "uhg29zk4": 2,
    "ich20c3q": 2,
    "7str7fhl": 2,
    "g8e83omk": 2,
    ## CATH, len 512 ##
    "q3m9fhii": 1,
    "5z4iaak9": 1,
    "3rs1hxky": 1,
    "2tjrgcde": 1,
    "v2cer77t": 1,
    "yfel5fnl": 1,
    "1hr1x9r5": 1,
    "1b64t79h": 1,
}

RESTYPES = [
    "A",
    "R",
    "N",
    "D",
    "C",
    "Q",
    "E",
    "G",
    "H",
    "I",
    "L",
    "K",
    "M",
    "F",
    "P",
    "S",
    "T",
    "W",
    "Y",
    "V",
]


UNIREF_SS_BOUNDARIES = {
    "helix": [0.1647, 0.3294, 0.4941, 0.6589, 0.8236],
    "turn": [0.1628, 0.3255, 0.4883, 0.6510, 0.8138],
    "sheet": [0.1524, 0.3048, 0.4571, 0.6095, 0.7619],
}


CATH_SS_BOUNDARIES = {
    "helix": [0.2109375, 0.25, 0.2734375, 0.296875, 0.3203125],
    "turn": [0.171875, 0.2109375, 0.234375, 0.265625, 0.3046875],
    "sheet": [0.1875, 0.2265625, 0.2578125, 0.296875, 0.3359375],
}

# for quantizing secondary structure fraction for conditioning
NUM_SECONDARY_STRUCTURE_BINS = 6

# for foldseek
PDB_DATABASE_PATH = "/data/bucket/robins21/pdb"

# for mmseqs
UNIREF_DATABASE_PATH = "/data/lux70/data/uniref50db/uniref50db"


# for mmseqs
FUNCTION_IDX_TO_GO_TERM = {
    "0": "G protein-coupled receptor activity",
    "1": "ATP hydrolysis activity",
    "2": "ATP binding",
    "3": "GTPase activity",
    "4": "protein dimerization activity",
    "5": "ATP-dependent protein folding chaperone",
    "6": "RNA binding",
    "7": "serine-type endopeptidase inhibitor activity",
    "8": "signal transduction",
    "9": "ribulose-bisphosphate carboxylase activity",
    "10": "protein binding",
    "11": "growth factor activity",
    "12": "homophilic cell adhesion via plasma membrane adhesion molecules",
    "13": "cysteine-type endopeptidase inhibitor activity",
    "14": "electron transfer activity",
    "15": "calcium ion binding",
    "16": "extracellular region",
    "17": "heme binding",
    "18": "NAD binding",
    "19": "DNA binding",
    "20": "chemokine activity",
    "21": "hormone activity",
    "22": "oxidoreductase activity",
    "23": "ligand-gated monoatomic ion channel activity",
    "24": "myosin complex",
    "25": "exo-alpha-sialidase activity",
    "26": "oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen",
    "27": "phospholipid metabolic process",
    "28": "protein kinase activity",
    "29": "phosphorelay signal transduction system",
    "30": "structural molecule activity",
    "31": "RNA-DNA hybrid ribonuclease activity",
    "32": "superoxide metabolic process",
    "33": "serine-type peptidase activity",
    "34": "transmembrane transporter activity",
    "35": "serine-type endopeptidase activity",
    "36": "GTP binding",
    "37": "peptidase inhibitor activity",
    "38": "metal ion binding",
    "39": "nucleic acid binding",
    "40": "protein dephosphorylation",
    "41": "sequence-specific DNA binding",
    "42": "acyltransferase activity, transferring groups other than amino-acyl groups",
    "43": "Wnt signaling pathway",
    "44": "iron-sulfur cluster binding",
    "45": "cysteine-type peptidase activity",
    "46": "pilus",
    "47": "aerobic respiration",
    "48": "cytochrome-c oxidase activity",
    "49": "proton-transporting ATP synthase complex, coupling factor F(o)",
    "50": "glutamine synthetase activity",
    "51": "triose-phosphate isomerase activity",
    "52": "photosynthetic electron transport in photosystem II",
    "53": "DNA-binding transcription factor activity",
    "54": "copper ion binding",
    "55": "carbohydrate metabolic process",
    "56": "tRNA aminoacylation for protein translation",
    "57": "nucleotide binding",
    "58": "proton-transporting two-sector ATPase complex, proton-transporting domain",
    "59": "carbohydrate binding",
    "60": "sigma factor activity",
    "61": "response to oxidative stress",
    "62": "cytokine receptor binding",
    "63": "methyltransferase activity",
    "64": "membrane",
    "65": "hydrolase activity",
    "66": "organic substance metabolic process",
    "67": "lipase activity",
    "68": "pyridoxal phosphate binding",
    "69": "transcription factor binding",
    "70": "protein peptidyl-prolyl isomerization",
    "71": "rRNA N-glycosylase activity",
    "72": "phosphoglycerate kinase activity",
    "73": "rRNA binding",
    "74": "ribosome",
    "75": "protein folding chaperone",
    "76": "DNA-binding transcription factor activity, RNA polymerase II-specific",
    "77": "ATP-dependent chromatin remodeler activity",
    "78": "chitinase activity",
    "79": "neurohypophyseal hormone activity",
    "80": "amino acid binding",
    "81": "tetrahydrofolate biosynthetic process",
    "82": "chitin binding",
    "83": "translation",
    "84": "calcium-dependent phospholipid binding",
    "85": "hyaluronic acid binding",
    "86": "regulation of DNA-templated transcription",
    "87": "endopeptidase inhibitor activity",
    "88": "acyltransferase activity",
    "89": "catalase activity",
    "90": "UDP-glycosyltransferase activity",
    "91": "transaminase activity",
    "92": "DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) activity",
    "93": "thiamine pyrophosphate binding",
    "94": "amino acid metabolic process",
    "95": "ferric iron binding",
    "96": "cyclic nucleotide biosynthetic process",
    "97": "proton-transporting ATP synthase activity, rotational mechanism",
    "98": "orotidine-5'-phosphate decarboxylase activity",
    "99": "structural constituent of chromatin",
    "100": "transferase activity, transferring phosphorus-containing groups",
    "101": "indole-3-glycerol-phosphate synthase activity",
    "102": "thylakoid",
    "103": "pyruvate kinase activity",
    "104": "microtubule motor activity",
    "105": "proteasome core complex",
    "106": "tumor necrosis factor receptor binding",
    "107": "channel activity",
    "108": "proton-transporting ATP synthase complex, catalytic core F(1)",
    "109": "hydrolase activity, hydrolyzing O-glycosyl compounds",
    "110": "3',5'-cyclic-nucleotide phosphodiesterase activity",
    "111": "actin binding",
    "112": "DNA strand exchange activity",
    "113": "DNA-directed DNA polymerase activity",
    "114": "signaling receptor binding",
    "115": "phosphatase activity",
    "116": "metallocarboxypeptidase activity",
    "117": "peptidyl-prolyl cis-trans isomerase activity",
    "118": "glutathione peroxidase activity",
    "119": "response to water",
    "120": "FMN binding",
    "121": "sperm DNA condensation",
    "122": "unfolded protein binding",
    "123": "protein secretion",
    "124": "thymidine kinase activity",
    "125": "monoatomic ion transmembrane transport",
    "126": "deoxyribonucleotide biosynthetic process",
    "127": "double-stranded DNA binding",
    "128": "antibacterial humoral response",
    "129": "extracellular space",
    "130": "fructose-bisphosphate aldolase activity",
    "131": "transferase activity, transferring alkyl or aryl (other than methyl) groups",
    "132": "catalytic activity",
    "133": "response to wounding",
    "134": "carbon-carbon lyase activity",
    "135": "photosynthesis",
    "136": "photosystem II",
    "137": "acyltransferase activity, acyl groups converted into alkyl on transfer",
    "138": "viral capsid",
    "139": "sodium:potassium-exchanging ATPase complex",
    "140": "tryptophan synthase activity",
    "141": "polygalacturonase activity",
    "142": "iron ion binding",
    "143": "chloramphenicol O-acetyltransferase activity",
    "144": "oxidoreductase activity, acting on single donors with incorporation of molecular oxygen, incorporation of two atoms of oxygen",
    "145": "proton motive force-driven ATP synthesis",
    "146": "DNA-binding transcription activator activity",
    "147": "phosphoenolpyruvate carboxylase activity",
    "148": "ribonucleoside-diphosphate reductase activity, thioredoxin disulfide as acceptor",
    "149": "defense response",
    "150": "regulation of vasoconstriction",
    "151": "transmembrane transport",
    "152": "NADH dehydrogenase (ubiquinone) activity",
    "153": "inflammatory response",
    "154": "glucose-6-phosphate isomerase activity",
    "155": "glycogen phosphorylase activity",
    "156": "protein transport",
    "157": "cell wall organization",
    "158": "oxidoreductase activity, acting on NAD(P)H",
    "159": "isoprenoid biosynthetic process",
    "160": "phosphotransferase activity, alcohol group as acceptor",
    "161": "oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced pteridine as one donor, and incorporation of one atom of oxygen",
    "162": "DNA-templated transcription initiation",
    "163": "2 iron, 2 sulfur cluster binding",
    "164": "phosphoenolpyruvate-dependent sugar phosphotransferase system",
    "165": "detection of visible light",
    "166": "6-phosphofructokinase activity",
    "167": "protein-N(PI)-phosphohistidine-sugar phosphotransferase activity",
    "168": "hydroxymethylglutaryl-CoA reductase (NADPH) activity",
    "169": "kinase activity",
    "170": "nickel cation binding",
    "171": "symporter activity",
    "172": "protein homooligomerization",
    "173": "TBP-class protein binding",
    "174": "phosphatidylinositol phospholipase C activity",
    "175": "oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor",
    "176": "malic enzyme activity",
    "177": "phosphorylation",
    "178": "pentose-phosphate shunt",
    "179": "structural constituent of cell wall",
    "180": "polysaccharide catabolic process",
    "181": "extracellular matrix",
    "182": "tubulin binding",
    "183": "photosystem",
    "184": "host cell surface receptor binding",
    "185": "host cell nucleus",
    "186": "viral life cycle",
    "187": "phycobilisome",
    "188": "protein phosphorylation",
    "189": "single-stranded DNA binding",
    "190": "S-adenosylmethionine biosynthetic process",
    "191": "oxidoreductase activity, acting on the CH-CH group of donors",
    "192": "protein deubiquitination",
    "193": "ribonuclease T2 activity",
    "194": "multicellular organism development",
    "195": "SRP-dependent cotranslational protein targeting to membrane",
    "196": "serine-type carboxypeptidase activity",
    "197": "ion channel inhibitor activity",
    "198": "isocitrate lyase activity",
    "199": "translation release factor activity",
    "200": "imidazoleglycerol-phosphate dehydratase activity",
    "201": "glucose metabolic process",
    "202": "biosynthetic process",
    "203": "carbonate dehydratase activity",
    "204": "lipid metabolic process",
    "205": "mismatched DNA binding",
    "206": "T-helper 17 cell lineage commitment",
    "207": "porphobilinogen synthase activity",
    "208": "DNA duplex unwinding",
    "209": "G-protein beta/gamma-subunit complex binding",
    "210": "regulation of DNA replication",
    "211": "phosphorelay sensor kinase activity",
    "212": "viral envelope",
    "213": "DNA helicase activity",
    "214": "monoatomic ion channel activity",
    "215": "fusion of virus membrane with host plasma membrane",
    "216": "hydrolase activity, acting on acid anhydrides",
    "217": "structural constituent of eye lens",
    "218": "glycosyltransferase activity",
    "219": "nucleosome assembly",
    "220": "RNA-binding transcription regulator activity",
    "221": "virion attachment to host cell",
    "222": "regulation of nitrogen utilization",
    "223": "RNA endonuclease activity",
    "224": "urea catabolic process",
    "225": "cysteine-type endopeptidase activity",
    "226": "plasma membrane light-harvesting complex",
    "227": "monoatomic cation channel activity",
    "228": "DNA-directed 5'-3' RNA polymerase activity",
    "229": "zinc ion binding",
    "230": "fimbrial usher porin activity",
    "231": "antioxidant activity",
    "232": "protein targeting",
    "233": "RNA methyltransferase activity",
    "234": "DNA integration",
    "235": "host cell membrane",
    "236": "exit of virus from host cell nucleus through nuclear pore",
    "237": "viral RNA genome replication",
    "238": "transcription cis-regulatory region binding",
    "239": "ATP-dependent diacylglycerol kinase activity",
    "240": "intracellular signal transduction",
    "241": "regulation of GTPase activity",
    "242": "small GTPase mediated signal transduction",
    "243": "regulation of apoptotic process",
    "244": "guanyl-nucleotide exchange factor activity",
    "245": "flocculation",
    "246": "G protein-coupled receptor signaling pathway",
    "247": "ubiquitin-protein transferase activity",
    "248": "ribonuclease III activity",
    "249": "vesicle-mediated transport",
    "250": "intracellular transport",
    "251": "NAD+ ADP-ribosyltransferase activity",
    "252": "translational elongation",
    "253": "calcium-dependent cysteine-type endopeptidase activity",
    "254": "voltage-gated chloride channel activity",
    "255": "hydrolase activity, acting on ester bonds",
    "256": "virion assembly",
    "257": "ABC-type transporter activity",
    "258": "oxidoreductase activity, acting on the aldehyde or oxo group of donors, disulfide as acceptor",
    "259": "RNA-dependent RNA polymerase activity",
    "260": "cytoskeleton",
    "261": "heat shock protein binding",
    "262": "sulfotransferase activity",
    "263": "starch binding",
    "264": "TMP biosynthetic process",
    "265": "viral process",
    "266": "phosphoribosylanthranilate isomerase activity",
    "267": "urease complex",
    "268": "lyase activity",
    "269": "translational initiation",
    "270": "adenylosuccinate synthase activity",
    "271": "DNA polymerase III complex",
    "272": "type II interferon receptor binding",
    "273": "interleukin-2 receptor binding",
    "274": "phosphate-containing compound metabolic process",
    "275": "fatty acid metabolic process",
    "276": "interleukin-4 receptor binding",
    "277": "base-excision repair",
    "278": "'de novo' IMP biosynthetic process",
    "279": "oxidoreductase activity, acting on CH-OH group of donors",
    "280": "asparagine synthase (glutamine-hydrolyzing) activity",
    "281": "cellulose binding",
    "282": "phosphate ion binding",
    "283": "viral genome replication",
    "284": "vesicle membrane",
    "285": "N,N-dimethylaniline monooxygenase activity",
    "286": "glutamyl-tRNA reductase activity",
    "287": "tRNA aminoacylation",
    "288": "nuclease activity",
    "289": "heme biosynthetic process",
    "290": "methylenetetrahydrofolate dehydrogenase (NADP+) activity",
    "291": "transferase activity",
    "292": "serine-type D-Ala-D-Ala carboxypeptidase activity",
    "293": "RNA nuclease activity",
    "294": "sialyltransferase activity",
    "295": "protein tyrosine/serine/threonine phosphatase activity",
    "296": "phosphatidylinositol binding",
    "297": "nitrogen compound metabolic process",
    "298": "photosystem I",
    "299": "acetyltransferase activity",
    "300": "DNA replication",
    "301": "L-phenylalanine biosynthetic process",
    "302": "virion membrane",
    "303": "transport of virus in host, cell to cell",
    "304": "pteridine-containing compound metabolic process",
    "305": "protein retention in ER lumen",
    "306": "cell-matrix adhesion",
    "307": "DNA repair",
    "308": "cell outer membrane",
    "309": "envenomation resulting in occlusion of the pore of voltage-gated potassium channel in another organism",
    "310": "phosphoenolpyruvate carboxykinase activity",
    "311": "ribonuclease P activity",
    "312": "structural constituent of ribosome",
    "313": "racemase and epimerase activity, acting on carbohydrates and derivatives",
    "314": "regulation of microtubule polymerization or depolymerization",
    "315": "thyroxine 5'-deiodinase activity",
    "316": "spermatid development",
    "317": "viral nucleocapsid",
    "318": "cellular metabolic process",
    "319": "RNA modification",
    "320": "fucosyltransferase activity",
    "321": "sodium channel activity",
    "322": "sucrose metabolic process",
    "323": "ossification",
    "324": "cellular aromatic compound metabolic process",
    "325": "peptide cross-linking",
    "326": "phosphotransferase activity, carboxyl group as acceptor",
    "327": "transposase activity",
    "328": "lysosomal transport",
    "329": "metalloaminopeptidase activity",
    "330": "riboflavin synthase complex",
    "331": "fatty-acyl-CoA binding",
    "332": "ubiquitin protein ligase binding",
    "333": "O-methyltransferase activity",
    "334": "mitochondrial proton-transporting ATP synthase complex, coupling factor F(o)",
    "335": "keratinization",
    "336": "penicillin binding",
    "337": "positive regulation of DNA-templated transcription",
    "338": "dTDP-4-dehydrorhamnose 3,5-epimerase activity",
    "339": "ammonium transmembrane transport",
    "340": "RNA helicase activity",
    "341": "evasion of host immune response",
    "342": "methylthiotransferase activity",
    "343": "hydro-lyase activity",
    "344": "3,4-dihydroxy-2-butanone-4-phosphate synthase activity",
    "345": "proteolysis",
    "346": "ADP binding",
    "347": "bacterial microcompartment",
    "348": "7-methylguanosine mRNA capping",
    "349": "peptidase activity",
    "350": "ATP-binding cassette (ABC) transporter complex",
    "351": "phosphotransferase activity, for other substituted phosphate groups",
    "352": "recognition of pollen",
    "353": "monoatomic anion transport",
    "354": "GMP biosynthetic process",
    "355": "cell wall macromolecule catabolic process",
    "356": "endonuclease activity",
    "357": "deaminase activity",
    "358": "metalloendopeptidase inhibitor activity",
    "359": "defense response to fungus",
    "360": "MHC class II protein complex",
    "361": "histidine biosynthetic process",
    "362": "trehalose biosynthetic process",
    "363": "cell adhesion",
    "364": "troponin complex",
    "365": "GDP-dissociation inhibitor activity",
    "366": "antiporter activity",
    "367": "ribonucleoside triphosphate phosphatase activity",
    "368": "virion component",
    "369": "double-stranded RNA binding",
    "370": "cytoplasm",
    "371": "pore-forming activity",
    "372": "adenyl-nucleotide exchange factor activity",
    "373": "DNA topoisomerase type I (single strand cut, ATP-independent) activity",
    "374": "protein import",
    "375": "actin filament binding",
    "376": "nucleoside metabolic process",
    "377": "polysaccharide metabolic process",
    "378": "DNA replication initiation",
    "379": "transsulfuration",
    "380": "cell surface",
    "381": "chloride channel activity",
    "382": "transmembrane receptor protein serine/threonine kinase activity",
    "383": "DNA ligase (ATP) activity",
    "384": "3-beta-hydroxy-delta5-steroid dehydrogenase activity",
    "385": "alpha-mannosidase activity",
    "386": "DNA recombination",
    "387": "protein autoprocessing",
    "388": "protein processing",
    "389": "oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced ascorbate as one donor, and incorporation of one atom of oxygen",
    "390": "cell-cell signaling",
    "391": "clathrin coat of coated pit",
    "392": "UDP-glucose:hexose-1-phosphate uridylyltransferase activity",
    "393": "cysteine-type deubiquitinase activity",
    "394": "cell wall modification",
    "395": "cell division",
    "396": "chromatin",
    "397": "outer membrane",
    "398": "iron-sulfur cluster assembly",
    "399": "granulocyte macrophage colony-stimulating factor receptor binding",
    "400": "ciliary neurotrophic factor-mediated signaling pathway",
    "401": "cyclin-dependent protein serine/threonine kinase regulator activity",
    "402": "4-hydroxy-tetrahydrodipicolinate reductase",
    "403": "digestion",
    "404": "F-actin capping protein complex",
    "405": "aldehyde-lyase activity",
    "406": "oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor",
    "407": "alpha-L-fucosidase activity",
    "408": "dephospho-CoA kinase activity",
    "409": "cobalamin transport",
    "410": "toxin activity",
    "411": "nucleus",
    "412": "heme oxygenase (decyclizing) activity",
    "413": "cytidylyltransferase activity",
    "414": "NAD+-protein-arginine ADP-ribosyltransferase activity",
    "415": "DNA topoisomerase activity",
    "416": "RNA ligase activity",
    "417": "pseudouridine synthase activity",
    "418": "CoA-transferase activity",
    "419": "caveola assembly",
    "420": "DNA N-glycosylase activity",
    "421": "fatty acid elongase activity",
    "422": "oxygen binding",
    "423": "collagen-containing extracellular matrix",
    "424": "protein maturation",
    "425": "hydrolase activity, hydrolyzing N-glycosyl compounds",
    "426": "neuropeptide signaling pathway",
    "427": "regulation of transcription by RNA polymerase II",
    "428": "pyridoxal phosphate biosynthetic process",
    "429": "racemase activity, acting on amino acids and derivatives",
    "430": "primary amine oxidase activity",
    "431": "fungal-type cell wall",
    "432": "oxidoreductase activity, acting on the CH-NH2 group of donors, oxygen as acceptor",
    "433": "aminoacyl-tRNA hydrolase activity",
    "434": "protein secretion by the type II secretion system",
    "435": "trehalose metabolic process",
    "436": "uroporphyrinogen decarboxylase activity",
    "437": "glycerol-3-phosphate catabolic process",
    "438": "cytoskeleton organization",
    "439": "protein kinase CK2 complex",
    "440": "mitochondrial envelope",
    "441": "coproporphyrinogen oxidase activity",
    "442": "phospholipid biosynthetic process",
    "443": "3-dehydroquinate dehydratase activity",
    "444": "dynein complex",
    "445": "oxidoreductase activity, acting on the CH-CH group of donors, NAD or NADP as acceptor",
    "446": "acid-amino acid ligase activity",
    "447": "glycylpeptide N-tetradecanoyltransferase activity",
    "448": "amino acid:sodium symporter activity",
    "449": "lipid binding",
    "450": "protein prenylation",
    "451": "metallodipeptidase activity",
    "452": "aspartic-type endopeptidase activity",
    "453": "nucleosome",
    "454": "isomerase activity",
    "455": "chorismate synthase activity",
    "456": "holocytochrome-c synthase activity",
    "457": "formate-tetrahydrofolate ligase activity",
    "458": "rRNA processing",
    "459": "secretory granule",
    "460": "regulation of DNA-templated transcription elongation",
    "461": "malate synthase activity",
    "462": "monolayer-surrounded lipid storage body",
    "463": "endopeptidase activity",
    "464": "positive regulation of translational termination",
    "465": "folic acid-containing compound biosynthetic process",
    "466": "cholesterol binding",
    "467": "actin monomer binding",
    "468": "cytokine activity",
    "469": "phosphoenolpyruvate carboxykinase (ATP) activity",
    "470": "cAMP biosynthetic process",
    "471": "metal ion transport",
    "472": "single fertilization",
    "473": "gas vesicle",
    "474": "carbohydrate:proton symporter activity",
    "475": "porin activity",
    "476": "hexon binding",
    "477": "oxidoreductase activity, acting on the aldehyde or oxo group of donors, iron-sulfur protein as acceptor",
    "478": "arginine metabolic process",
    "479": "toxic substance binding",
    "480": "peptide deformylase activity",
    "481": "peroxidase activity",
    "482": "4-alpha-hydroxytetrahydrobiopterin dehydratase activity",
    "483": "four-way junction helicase activity",
    "484": "mRNA guanylyltransferase activity",
    "485": "thylakoid membrane",
    "486": "protein-glutamate methylesterase activity",
    "487": "methionine metabolic process",
    "488": "cellulose catabolic process",
    "489": "protein folding",
    "490": "lipid transporter activity",
    "491": "mRNA processing",
    "492": "RNA metabolic process",
    "493": "bioluminescence",
    "494": "aerobic electron transport chain",
    "495": "alpha-amylase inhibitor activity",
    "496": "calcium ion transmembrane transport",
    "497": "viral DNA genome packaging",
    "498": "regulation of ARF protein signal transduction",
    "499": "protein kinase inhibitor activity",
    "500": "beta-amylase activity",
    "501": "chitosanase activity",
    "502": "cell wall",
    "503": "hydroxymethylbilane synthase activity",
    "504": "carbohydrate derivative binding",
    "505": "biotin binding",
    "506": "inorganic phosphate transmembrane transporter activity",
    "507": "odorant binding",
    "508": "chromosome",
    "509": "terpene synthase activity",
    "510": "metallopeptidase activity",
    "511": "metalloendopeptidase activity",
    "512": "peptidyl-dipeptidase activity",
    "513": "photosystem II reaction center",
    "514": "extracellular matrix structural constituent",
    "515": "alanyl-tRNA aminoacylation",
    "516": "GTPase activator activity",
    "517": "collagen trimer",
    "518": "cell communication",
    "519": "interleukin-7 receptor binding",
    "520": "chromatin binding",
    "521": "dipeptidase activity",
    "522": "ATP-dependent peptidase activity",
    "523": "lipoprotein metabolic process",
    "524": "ketol-acid reductoisomerase activity",
    "525": "mitochondrial outer membrane",
    "526": "ubiquitin-dependent protein catabolic process",
    "527": "3-deoxy-7-phosphoheptulonate synthase activity",
    "528": "structural constituent of cuticle",
    "529": "FAD binding",
    "530": "proton-transporting V-type ATPase, V0 domain",
    "531": "viral genome packaging",
    "532": "keratin filament",
    "533": "phosphoribosyl-AMP cyclohydrolase activity",
    "534": "phosphatidylinositol kinase activity",
    "535": "RNA processing",
    "536": "N-acetylmuramoyl-L-alanine amidase activity",
    "537": "NADP biosynthetic process",
    "538": "viral entry into host cell",
    "539": "single-stranded RNA binding",
    "540": "hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds",
    "541": "negative regulation of gene silencing by regulatory ncRNA",
    "542": "host cell cytoplasm",
    "543": "palmitoyltransferase activity",
    "544": "galactoside 2-alpha-L-fucosyltransferase activity",
    "545": "mannosyl-oligosaccharide 1,2-alpha-mannosidase activity",
    "546": "cell surface receptor signaling pathway",
    "547": "metal ion transmembrane transporter activity",
    "548": "monoatomic cation transmembrane transporter activity",
    "549": "xenobiotic transmembrane transporter activity",
    "550": "DNA methylation",
    "551": "oxidoreductase activity, acting on the aldehyde or oxo group of donors",
    "552": "nutrient reservoir activity",
    "553": "virus-mediated perturbation of host defense response",
    "554": "flavin adenine dinucleotide binding",
    "555": "molybdopterin cofactor binding",
    "556": "host cell junction",
    "557": "cytochrome complex assembly",
    "558": "chemotaxis",
    "559": "tRNA binding",
    "560": "fructose metabolic process",
    "561": "receptor-mediated virion attachment to host cell",
    "562": "membrane coat",
    "563": "protein phosphatase type 2A complex",
    "564": "nucleobase-containing compound metabolic process",
    "565": "peptide-methionine (S)-S-oxide reductase activity",
    "566": "hyalurononglucosaminidase activity",
    "567": "ATP phosphoribosyltransferase activity",
    "568": "structural constituent of virion",
    "569": "peptide-methionine (R)-S-oxide reductase activity",
    "570": "intramolecular transferase activity",
    "571": "thiolester hydrolase activity",
    "572": "chitin biosynthetic process",
    "573": "glutamate biosynthetic process",
    "574": "holo-[acyl-carrier-protein] synthase activity",
    "575": "DNA ligase (NAD+) activity",
    "576": "cytochrome complex",
    "577": "mRNA methyltransferase activity",
    "578": "structural constituent of myelin sheath",
    "579": "taxis",
    "580": "lipid catabolic process",
    "581": "diaminopimelate epimerase activity",
    "582": "T=25 icosahedral viral capsid",
    "583": "riboflavin kinase activity",
    "584": "host cell",
    "585": "suppression by virus of host apoptotic process",
    "586": "pentosyltransferase activity",
    "587": "uridylyltransferase activity",
    "588": "oxygen evolving activity",
    "589": "5-methyltetrahydropteroyltriglutamate-homocysteine S-methyltransferase activity",
    "590": "extrachromosomal circular DNA",
    "591": "defense response to bacterium",
    "592": "structural constituent of egg chorion",
    "593": "nucleoside triphosphate catabolic process",
    "594": "methylation",
    "595": "nicotinate-nucleotide diphosphorylase (carboxylating) activity",
    "596": "arylesterase activity",
    "597": "nucleoside transmembrane transport",
    "598": "phospholipid catabolic process",
    "599": "photosystem II stabilization",
    "600": "nucleotidyltransferase activity",
    "601": "sulfate adenylyltransferase (ATP) activity",
    "602": "nuclear import signal receptor activity",
    "603": "enzyme activator activity",
    "604": "fatty acid beta-oxidation",
    "605": "hexosyltransferase activity",
    "606": "chromosome organization",
    "607": "vitamin transport",
    "608": "large ribosomal subunit",
    "609": "alternative oxidase activity",
    "610": "extrinsic component of membrane",
    "611": "phosphatidylglycerol-prolipoprotein diacylglyceryl transferase activity",
    "612": "mannosyltransferase activity",
    "613": "antibiotic biosynthetic process",
    "614": "DNA primase activity",
    "615": "phosphoribosylaminoimidazolecarboxamide formyltransferase activity",
    "616": "amino acid transport",
    "617": "proton-transporting ATPase activity, rotational mechanism",
    "618": "chorismate metabolic process",
    "619": "glutamic-type endopeptidase activity",
    "620": "amidase activity",
    "621": "single strand break repair",
    "622": "protein serine/threonine kinase activator activity",
    "623": "DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) inhibitor activity",
    "624": "histone acetyltransferase activity",
    "625": "regulation of cell cycle",
    "626": "arginine decarboxylase activity",
    "627": "protein histidyl modification to diphthamide",
    "628": "defense response to virus",
    "629": "ribonuclease P complex",
    "630": "5-amino-6-(5-phosphoribosylamino)uracil reductase activity",
    "631": "triphosphoribosyl-dephospho-CoA synthase activity",
    "632": "tRNA processing",
    "633": "cobalamin biosynthetic process",
    "634": "transition metal ion transport",
    "635": "DNA replication, synthesis of RNA primer",
    "636": "cytosolic ribosome assembly",
    "637": "one-carbon metabolic process",
    "638": "peptidyl-lysine modification to peptidyl-hypusine",
    "639": "archaeal or bacterial-type flagellum-dependent cell motility",
    "640": "prefoldin complex",
    "641": "lysyl-tRNA aminoacylation",
    "642": "signal recognition particle",
    "643": "LPPG:FO 2-phospho-L-lactate transferase activity",
    "644": "toxin-antitoxin complex",
    "645": "aspartate dehydrogenase activity",
    "646": "3-dehydroquinate synthase activity",
    "647": "arginine biosynthetic process",
    "648": "thiamine biosynthetic process",
    "649": "Mo-molybdopterin cofactor biosynthetic process",
    "650": "tRNA-intron endonuclease activity",
    "651": "phospholactate guanylyltransferase activity",
    "652": "intracellular manganese ion homeostasis",
    "653": "proton-transporting two-sector ATPase complex, catalytic domain",
    "654": "ferredoxin hydrogenase activity",
    "655": "tRNA nucleoside ribose methylation",
    "656": "tRNA (guanine(10)-N2)-methyltransferase activity",
    "657": "cellulase activity",
    "658": "apoptotic process",
    "659": "hydrolase activity, acting on glycosyl bonds",
    "660": "interleukin-5 receptor binding",
    "661": "nitrogen compound transport",
    "662": "hemolymph coagulation",
    "663": "regulation of monoatomic ion transport",
    "664": "arsenite transport",
    "665": "auxin binding",
    "666": "mitochondrial respiratory chain complex IV",
    "667": "bacterial-type flagellum",
    "668": "guanylate cyclase activator activity",
    "669": "interleukin-3 receptor binding",
    "670": "voltage-gated potassium channel activity",
    "671": "calmodulin binding",
    "672": "regulation of smooth muscle contraction",
    "673": "feeding behavior",
    "674": "aminopeptidase activity",
    "675": "DNA endonuclease activity",
    "676": "mating-type factor pheromone receptor activity",
    "677": "blood vessel diameter maintenance",
    "678": "fusion of sperm to egg plasma membrane involved in single fertilization",
    "679": "DNA methylation on adenine",
    "680": "histamine binding",
    "681": "regulation of blood coagulation",
    "682": "glycyl-tRNA aminoacylation",
    "683": "glycine-tRNA ligase activity",
    "684": "amine binding",
    "685": "ornithine decarboxylase inhibitor activity",
    "686": "tyrosine binding",
    "687": "interstrand cross-link repair",
    "688": "bacterial-type flagellum basal body, distal rod, L ring",
    "689": "oligosaccharyltransferase complex",
    "690": "hydroxyethylthiazole kinase activity",
    "691": "3',5'-cyclic-AMP phosphodiesterase activity",
    "692": "Rho protein signal transduction",
    "693": "sensory perception of chemical stimulus",
    "694": "bacterial-type flagellum basal body, distal rod, P ring",
    "695": "phospholipid transporter activity",
    "696": "catabolic process",
    "697": "adenosine deaminase activity",
    "698": "DNA damage checkpoint signaling",
    "699": "regulation of small GTPase mediated signal transduction",
    "700": "NAD+ binding",
    "701": "DNA-templated transcription",
    "702": "folic acid-containing compound metabolic process",
    "703": "bacterial-type flagellum basal body",
    "704": "glucocorticoid receptor signaling pathway",
    "705": "nuclear estrogen receptor activity",
    "706": "nuclear steroid receptor activity",
    "707": "androgen receptor signaling pathway",
    "708": "transcription coregulator activity",
    "709": "heparin binding",
    "710": "protein-folding chaperone binding",
    "711": "transcription initiation at RNA polymerase II promoter",
    "712": "microtubule binding",
    "713": "GTPase regulator activity",
    "714": "tachykinin receptor signaling pathway",
    "715": "immunoglobulin binding",
    "716": "DNA replication origin binding",
    "717": "methylenetetrahydrofolate reductase (NAD(P)H) activity",
    "718": "cytidylate kinase activity",
    "719": "cyclin-dependent protein serine/threonine kinase inhibitor activity",
    "720": "DNA unwinding involved in DNA replication",
    "721": "protein modification process",
    "722": "coenzyme-B sulfoethylthiotransferase activity",
    "723": "alkylbase DNA N-glycosylase activity",
    "724": "host cell plasmodesma",
    "725": "proteasome activator complex",
    "726": "phospholipase activity",
    "727": "potassium ion transport",
    "728": "hemolysis by symbiont of host erythrocytes",
    "729": "alanine biosynthetic process",
    "730": "carbohydrate transmembrane transport",
    "731": "DNA catabolic process",
    "732": "NAD+ nucleosidase activity",
    "733": "transcription factor TFIIA complex",
    "734": "transcription by RNA polymerase II",
    "735": "mitochondrial electron transport, ubiquinol to cytochrome c",
    "736": "nicotinate-nucleotide-dimethylbenzimidazole phosphoribosyltransferase activity",
    "737": "adenosylcobinamide kinase activity",
    "738": "mitochondrial electron transport, cytochrome c to oxygen",
    "739": "cobalamin binding",
    "740": "methenyltetrahydromethanopterin cyclohydrolase activity",
    "741": "signal recognition particle, endoplasmic reticulum targeting",
    "742": "double-stranded RNA adenosine deaminase activity",
    "743": "clathrin adaptor complex",
    "744": "viral capsid assembly",
    "745": "modulation by virus of host process",
    "746": "transcription coactivator activity",
    "747": "fumarate metabolic process",
    "748": "alcohol dehydrogenase (NAD+) activity",
    "749": "transcription regulator complex",
    "750": "efflux transmembrane transporter activity",
    "751": "glucan biosynthetic process",
    "752": "histidine decarboxylase activity",
    "753": "mitochondrial matrix",
    "754": "cysteine-type endopeptidase inhibitor activity involved in apoptotic process",
    "755": "3-phytase activity",
    "756": "DNA replication termination",
    "757": "nitrite reductase (cytochrome, ammonia-forming) activity",
    "758": "ribulose bisphosphate carboxylase complex assembly",
    "759": "transcription elongation-coupled chromatin remodeling",
    "760": "1,3-beta-D-glucan synthase complex",
    "761": "protein O-linked glycosylation",
    "762": "tRNA threonylcarbamoyladenosine modification",
    "763": "plasmid maintenance",
    "764": "aminoacyltransferase activity",
    "765": "tRNA (guanine(46)-N7)-methyltransferase activity",
    "766": "interleukin-1 receptor binding",
    "767": "dimethylallyl diphosphate biosynthetic process",
    "768": "cytolysis",
    "769": "stem cell factor receptor binding",
    "770": "monooxygenase activity",
    "771": "endodeoxyribonuclease activity, producing 5'-phosphomonoesters",
    "772": "mercury ion transmembrane transporter activity",
    "773": "chromate transmembrane transporter activity",
    "774": "intermediate filament",
    "775": "protein flavinylation",
    "776": "photosystem I reaction center",
    "777": "light-harvesting complex",
    "778": "carbohydrate utilization",
    "779": "suppression by virus of host gene expression",
    "780": "quinolinate synthetase A activity",
    "781": "4-alpha-glucanotransferase activity",
    "782": "gluconate transmembrane transporter activity",
    "783": "beta-galactosidase complex",
    "784": "O-acyltransferase activity",
    "785": "bacterial-type flagellum hook",
    "786": "cytosol",
    "787": "regulation of gene expression",
    "788": "DNA-mediated transformation",
    "789": "primary metabolic process",
    "790": "fatty acid biosynthetic process",
    "791": "methanogenesis",
    "792": "nucleoside diphosphate kinase activity",
    "793": "dTMP biosynthetic process",
    "794": "gamete generation",
    "795": "CAAX-box protein processing",
    "796": "response to auxin",
    "797": "aminoglycoside 3-N-acetyltransferase activity",
    "798": "rRNA methyltransferase activity",
    "799": "cytochrome b6f complex",
    "800": "2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase activity",
    "801": "nucleoside triphosphate diphosphatase activity",
    "802": "3-methyl-2-oxobutanoate hydroxymethyltransferase activity",
    "803": "methanogenesis, from acetate",
    "804": "cobalt ion transmembrane transporter activity",
    "805": "cellular response to starvation",
    "806": "protein tetramerization",
    "807": "cyanate metabolic process",
    "808": "bacterial-type flagellum assembly",
    "809": "CCA tRNA nucleotidyltransferase activity",
    "810": "pantoate-beta-alanine ligase activity",
    "811": "precorrin-8X methylmutase activity",
    "812": "precorrin-6A reductase activity",
    "813": "corrinoid adenosyltransferase activity",
    "814": "D-aminoacyl-tRNA deacylase activity",
    "815": "protein-DNA covalent cross-linking",
    "816": "queuosine salvage",
    "817": "organic acid phosphorylation",
    "818": "regulation of carbohydrate metabolic process",
    "819": "protein-disulfide reductase activity",
    "820": "exodeoxyribonuclease VII activity",
    "821": "uroporphyrinogen-III synthase activity",
    "822": "tetraacyldisaccharide 4'-kinase activity",
    "823": "plasma membrane",
    "824": "exodeoxyribonuclease VII complex",
    "825": "L-arabinose isomerase activity",
    "826": "CDP-diacylglycerol diphosphatase activity",
    "827": "glucuronate isomerase activity",
    "828": "protein catabolic process",
    "829": "peroxiredoxin activity",
    "830": "oxidoreductase activity, acting on NAD(P)H, heme protein as acceptor",
    "831": "regulation of DNA repair",
    "832": "biotin transport",
    "833": "sulfurtransferase activity",
    "834": "carbon-nitrogen ligase activity, with glutamine as amido-N-donor",
    "835": "GTP cyclohydrolase activity",
    "836": "lactate transport",
    "837": "adenosylcobinamide-GDP ribazoletransferase activity",
    "838": "acyl-phosphate glycerol-3-phosphate acyltransferase activity",
    "839": "S-ribosylhomocysteine lyase activity",
    "840": "phosphatidylserine decarboxylase activity",
    "841": "P-type potassium transmembrane transporter activity",
    "842": "NADPH binding",
    "843": "undecaprenyl-diphosphatase activity",
    "844": "toxin biosynthetic process",
    "845": "spermidine biosynthetic process",
    "846": "lipid-A-disaccharide synthase activity",
    "847": "glucokinase activity",
    "848": "regulation of translational fidelity",
    "849": "helicase activity",
    "850": "sodium-dependent phosphate transport",
    "851": "protein histidine kinase activity",
    "852": "potassium ion transmembrane transporter activity",
    "853": "exit of virus from host cell nucleus by nuclear egress",
    "854": "viral budding from Golgi membrane",
    "855": "carboxyl- or carbamoyltransferase activity",
    "856": "spliceosomal complex",
    "857": "glycerone kinase activity",
    "858": "double-strand break repair via nonhomologous end joining",
    "859": "transition metal ion binding",
    "860": "chromosome, telomeric region",
    "861": "amino acid biosynthetic process",
    "862": "glucose-6-phosphate dehydrogenase activity",
    "863": "electron transport chain",
    "864": "exon-exon junction complex",
    "865": "toxin metabolic process",
    "866": "cation binding",
    "867": "plasminogen activation",
    "868": "negative regulation of protein kinase activity",
    "869": "small molecule binding",
    "870": "pyrophosphatase activity",
    "871": "RNA splicing",
    "872": "purine nucleobase biosynthetic process",
    "873": "ubiquitin binding",
    "874": "arginine binding",
    "875": "methylated-DNA-[protein]-cysteine S-methyltransferase activity",
    "876": "nucleotide catabolic process",
    "877": "UDP-N-acetylmuramate dehydrogenase activity",
    "878": "ATP metabolic process",
    "879": "intramolecular phosphotransferase activity",
    "880": "calcium-activated potassium channel activity",
    "881": "nitric oxide biosynthetic process",
    "882": "ferrous iron binding",
    "883": "transformation of host cell by virus",
    "884": "negative regulation of DNA-templated transcription",
    "885": "phenylalanine-tRNA ligase activity",
    "886": "integrase activity",
    "887": "ubiquinol-cytochrome-c reductase activity",
    "888": "viral procapsid maturation",
    "889": "extracellular ligand-gated monoatomic ion channel activity",
    "890": "monoatomic ion transport",
    "891": "ligase activity",
    "892": "polynucleotide 5'-phosphatase activity",
    "893": "oxidoreductase activity, acting on iron-sulfur proteins as donors",
    "894": "sensory perception of smell",
    "895": "glutathione synthase activity",
    "896": "fucose metabolic process",
    "897": "5-carboxymethyl-2-hydroxymuconate delta-isomerase activity",
    "898": "type II site-specific deoxyribonuclease activity",
    "899": "methane monooxygenase activity",
    "900": "methionine synthase activity",
    "901": "U4/U6 x U5 tri-snRNP complex",
    "902": "post-chaperonin tubulin folding pathway",
    "903": "folic acid binding",
    "904": "oxidoreductase activity, acting on the CH-NH2 group of donors",
    "905": "DNA restriction-modification system",
    "906": "melanin metabolic process",
    "907": "phospholipase inhibitor activity",
    "908": "lysis of host organelle involved in viral entry into host cell",
    "909": "diacylglycerol O-acyltransferase activity",
    "910": "phosphoric diester hydrolase activity",
    "911": "viral transcription",
    "912": "minus-end-directed microtubule motor activity",
    "913": "diphosphate hydrolysis-driven proton transmembrane transporter activity",
    "914": "lipid glycosylation",
    "915": "L-serine-phosphatidylethanolamine phosphatidyltransferase activity",
    "916": "positive regulation of cell population proliferation",
    "917": "interleukin-12 receptor binding",
    "918": "pheromone activity",
    "919": "cysteine-type aminopeptidase activity",
    "920": "nicotianamine biosynthetic process",
    "921": "nitronate monooxygenase activity",
    "922": "protein-arginine deiminase activity",
    "923": "hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in linear amides",
    "924": "acetylglucosaminyltransferase activity",
    "925": "glutamate-cysteine ligase activity",
    "926": "acireductone dioxygenase [iron(II)-requiring] activity",
    "927": "phosphatidylinositol-4,5-bisphosphate binding",
    "928": "mating",
    "929": "root development",
    "930": "strictosidine synthase activity",
    "931": "response to metal ion",
    "932": "phosphatase activator activity",
    "933": "protein-heme linkage",
    "934": "carbohydrate biosynthetic process",
    "935": "viral tegument",
    "936": "proteasomal protein catabolic process",
    "937": "nitrogenase activity",
    "938": "laminin binding",
    "939": "exonuclease activity",
    "940": "oligopeptide transmembrane transporter activity",
    "941": "UDP-glucose metabolic process",
    "942": "vacuolar proton-transporting V-type ATPase complex",
    "943": "threonine-phosphate decarboxylase activity",
    "944": "proton channel activity",
    "945": "mRNA splice site recognition",
    "946": "mercury ion transport",
    "947": "nitrogen fixation",
    "948": "pectate lyase activity",
    "949": "ATP:ADP antiporter activity",
    "950": "amino acid transmembrane transport",
    "951": "proton-transporting V-type ATPase, V1 domain",
    "952": "vacuolar proton-transporting V-type ATPase, V1 domain",
    "953": "oxidoreductase activity, acting on a sulfur group of donors, disulfide as acceptor",
    "954": "protein kinase binding",
    "955": "high-affinity iron permease complex",
    "956": "alkylmercury lyase activity",
    "957": "viral release from host cell by cytolysis",
    "958": "tropomyosin binding",
    "959": "urea transmembrane transporter activity",
    "960": "acetyl-CoA carboxylase complex",
    "961": "protein kinase 5 complex",
    "962": "deoxyribonuclease II activity",
    "963": "UDP-galactopyranose mutase activity",
    "964": "host cytoskeleton",
    "965": "regulation of cell shape",
    "966": "poly(A) RNA polymerase activity",
    "967": "tryptophan 2,3-dioxygenase activity",
    "968": "meiotic drive",
    "969": "acetolactate decarboxylase activity",
    "970": "pantothenate kinase activity",
    "971": "L-serine ammonia-lyase activity",
    "972": "fructose 1,6-bisphosphate 1-phosphatase activity",
    "973": "spore germination",
    "974": "DNA polymerase processivity factor activity",
    "975": "UDP-3-O-[3-hydroxymyristoyl] N-acetylglucosamine deacetylase activity",
    "976": "GDP-mannose biosynthetic process",
    "977": "inorganic cation transmembrane transport",
    "978": "response to antibiotic",
    "979": "mRNA splicing, via spliceosome",
    "980": "protein N-linked glycosylation via asparagine",
    "981": "DNA-3-methyladenine glycosylase activity",
    "982": "vacuolar transport",
    "983": "signaling",
    "984": "galactosylgalactosylxylosylprotein 3-beta-glucuronosyltransferase activity",
    "985": "RNA endonuclease activity, producing 5'-phosphomonoesters",
    "986": "small GTPase binding",
    "987": "heme transmembrane transporter activity",
    "988": "organic anion transport",
    "989": "pheromone receptor activity",
    "990": "molybdenum ion binding",
    "991": "acyl-[acyl-carrier-protein] desaturase activity",
    "992": "outer membrane-bounded periplasmic space",
    "993": "protein-phosphatidylethanolamide deconjugating activity",
    "994": "asexual sporulation",
    "995": "chitin catabolic process",
    "996": "[protein-PII] uridylyltransferase activity",
    "997": "NADP binding",
    "998": "molybdopterin cofactor biosynthetic process",
    "999": "translational termination",
    "1000": "regulatory ncRNA-mediated gene silencing",
    "1001": "phenylalanyl-tRNA aminoacylation",
    "1002": "arginyl-tRNA aminoacylation",
    "1003": "serotonin:sodium:chloride symporter activity",
    "1004": "calcium- and calmodulin-responsive adenylate cyclase activity",
    "1005": "cellulose biosynthetic process",
    "1006": "connexin complex",
    "1007": "xylan 1,4-beta-xylosidase activity",
    "1008": "voltage-gated potassium channel complex",
    "1009": "receptor-mediated endocytosis",
    "1010": "reciprocal meiotic recombination",
    "1011": "small conductance calcium-activated potassium channel activity",
    "1012": "positive regulation of GTPase activity",
    "1013": "cellulose synthase (UDP-forming) activity",
    "1014": "allantoicase activity",
    "1015": "heterocyst development",
    "1016": "cysteine-type exopeptidase activity",
    "1017": "triglyceride lipase activity",
    "1018": "deNEDDylase activity",
    "1019": "rRNA (pseudouridine) methyltransferase activity",
    "1020": "leucyl-tRNA--protein transferase activity",
    "1021": "aspartate-ammonia ligase activity",
    "1022": "benzoate transmembrane transporter activity",
    "1023": "carbon-monoxide dehydrogenase (acceptor) activity",
    "1024": "3'-5' exonuclease activity",
    "1025": "C4-dicarboxylate transmembrane transporter activity",
    "1026": "glutamate:sodium symporter activity",
    "1027": "protein tyrosine kinase activity",
    "1028": "retromer, cargo-selective complex",
    "1029": "intracellular protein transport",
    "1030": "mannosyl-glycoprotein endo-beta-N-acetylglucosaminidase activity",
    "1031": "xylan catabolic process",
    "1032": "mitochondrial pyruvate transmembrane transport",
    "1033": "glucan endo-1,3-alpha-glucosidase activity",
    "1034": "L-arabinose metabolic process",
    "1035": "EMC complex",
    "1036": "negative regulation of TOR signaling",
    "1037": "protein insertion into ER membrane",
    "1038": "protein ufmylation",
    "1039": "amino sugar metabolic process",
    "1040": "carboxy-lyase activity",
    "1041": "[glutamate-ammonia-ligase] adenylyltransferase activity",
    "1042": "pyridoxine 5'-phosphate synthase activity",
    "1043": "electron transporter, transferring electrons within cytochrome b6/f complex of photosystem II activity",
    "1044": "biotin biosynthetic process",
    "1045": "embryo development ending in seed dormancy",
    "1046": "inositol phosphate biosynthetic process",
    "1047": "cell morphogenesis",
    "1048": "regulation of division septum assembly",
    "1049": "mannonate dehydratase activity",
    "1050": "Ndc80 complex",
    "1051": "attachment of mitotic spindle microtubules to kinetochore",
    "1052": "prosthetic group biosynthetic process",
    "1053": "plasma membrane phospholipid scrambling",
    "1054": "adhesion of symbiont to microvasculature",
    "1055": "p-aminobenzoyl-glutamate transmembrane transport",
    "1056": "DNA transposition",
    "1057": "2-keto-3-deoxygluconate:proton symporter activity",
    "1058": "monoatomic ion transmembrane transporter activity",
    "1059": "nickel cation transmembrane transporter activity",
    "1060": "nucleoside transport",
    "1061": "orexin receptor activity",
    "1062": "DNA metabolic process",
    "1063": "endoplasmic reticulum membrane",
    "1064": "protein-transporting ATPase activity",
    "1065": "silicic acid import across plasma membrane",
    "1066": "negative regulation of cell division",
    "1067": "transcription factor TFIID complex",
    "1068": "ATPase activator activity",
    "1069": "transcription factor TFIIH core complex",
    "1070": "response to UV",
    "1071": "mismatch repair",
    "1072": "mating pheromone activity",
    "1073": "virion attachment to host cell pilus",
    "1074": "sigma factor antagonist activity",
    "1075": "RNA polymerase complex",
    "1076": "regulation of bone mineralization",
    "1077": "endoplasmic reticulum to Golgi vesicle-mediated transport",
    "1078": "rescue of stalled ribosome",
    "1079": "anaerobic respiration",
    "1080": "protein phosphatase regulator activity",
    "1081": "SNAP receptor activity",
    "1082": "photosystem II oxygen evolving complex",
    "1083": "mRNA transport",
    "1084": "killing by symbiont of host cells",
    "1085": "glutamine biosynthetic process",
    "1086": "L-lysine efflux transmembrane transporter activity",
    "1087": "egg chorion",
    "1088": "electron transporter, transferring electrons within the cyclic electron transport pathway of photosynthesis activity",
    "1089": "isocitrate dehydrogenase (NADP+) activity",
    "1090": "perturbation of host defenses by symbiont",
    "1091": "protein-glutamine glutaminase activity",
    "1092": "sodium ion transport",
    "1093": "MIS12/MIND type complex",
    "1094": "protein-macromolecule adaptor activity",
    "1095": "Cdc73/Paf1 complex",
    "1096": "ubiquitin-like protein transferase activity",
    "1097": "endosome transport via multivesicular body sorting pathway",
    "1098": "small-subunit processome",
    "1099": "checkpoint clamp complex",
    "1100": "Mpp10 complex",
    "1101": "tRNA methyltransferase activity",
    "1102": "coenzyme A biosynthetic process",
    "1103": "sporulation resulting in formation of a cellular spore",
    "1104": "2-phosphosulfolactate phosphatase activity",
    "1105": "D-arabinono-1,4-lactone oxidase activity",
    "1106": "Las1 complex",
    "1107": "enzyme inhibitor activity",
    "1108": "actin cytoskeleton",
    "1109": "vesicle docking involved in exocytosis",
    "1110": "anaphase-promoting complex",
    "1111": "periplasmic space",
    "1112": "Arp2/3 protein complex",
    "1113": "aminoacyl-tRNA editing activity",
    "1114": "tRNA wobble position uridine thiolation",
    "1115": "CCR4-NOT complex",
    "1116": "chromosome separation",
    "1117": "GPI anchor biosynthetic process",
    "1118": "DNA synthesis involved in DNA repair",
    "1119": "origin recognition complex",
    "1120": "signal recognition particle receptor complex",
    "1121": "protein import into peroxisome matrix, docking",
    "1122": "RNA polymerase I core promoter sequence-specific DNA binding",
    "1123": "nucleocytoplasmic transport",
    "1124": "structural constituent of nuclear pore",
    "1125": "TRAPP complex",
    "1126": "cellular modified amino acid biosynthetic process",
    "1127": "autophagy",
    "1128": "autophagosome assembly",
    "1129": "NatC complex",
    "1130": "attachment of GPI anchor to protein",
    "1131": "GPI-anchor transamidase complex",
    "1132": "ureidoglycolate lyase activity",
    "1133": "lipid biosynthetic process",
    "1134": "Golgi transport complex",
    "1135": "gamma-tubulin binding",
    "1136": "N-acylglucosamine-6-phosphate 2-epimerase activity",
    "1137": "snoRNA binding",
    "1138": "protein folding in endoplasmic reticulum",
    "1139": "polysaccharide biosynthetic process",
    "1140": "C-terminal protein methylation",
    "1141": "pyrimidine nucleotide-sugar transmembrane transporter activity",
    "1142": "copper ion transmembrane transport",
    "1143": "COPII receptor activity",
    "1144": "double-strand break repair",
    "1145": "intracellular sterol transport",
    "1146": "karyogamy involved in conjugation with cellular fusion",
    "1147": "regulation of phosphoprotein phosphatase activity",
    "1148": "regulation of signal transduction",
    "1149": "ribosomal small subunit biogenesis",
    "1150": "siderophore biosynthetic process",
    "1151": "alpha-1,6-mannosyltransferase activity",
    "1152": "tRNA (m1A) methyltransferase complex",
    "1153": "arylformamidase activity",
    "1154": "tetrahydromethanopterin S-methyltransferase activity",
    "1155": "ATP-independent citrate lyase complex",
    "1156": "peptidoglycan binding",
    "1157": "pseudouridylate synthase activity",
    "1158": "response to copper ion",
    "1159": "nucleoid",
    "1160": "aspartic endopeptidase activity, intramembrane cleaving",
    "1161": "thiamine diphosphokinase activity",
    "1162": "thiamine binding",
    "1163": "sarcosine oxidase activity",
    "1164": "regulation of ATPase-coupled calcium transmembrane transporter activity",
    "1165": "cholesterol biosynthetic process",
    "1166": "sodium ion transmembrane transporter activity",
    "1167": "chromosome condensation",
    "1168": "glycerol metabolic process",
    "1169": "DNA modification",
    "1170": "[acyl-carrier-protein] phosphodiesterase activity",
    "1171": "regulation of chemotaxis",
    "1172": "chorismate lyase activity",
    "1173": "ethanolamine transport",
    "1174": "bacterial-type flagellum organization",
    "1175": "type IV pilus-dependent motility",
    "1176": "cell septum assembly",
    "1177": "putrescine biosynthetic process",
    "1178": "arginyl-tRNA--protein transferase activity",
    "1179": "protein arginylation",
    "1180": "rRNA base methylation",
    "1181": "cortical actin cytoskeleton organization",
    "1182": "Gram-negative-bacterium-type cell outer membrane assembly",
    "1183": "ribosome biogenesis",
    "1184": "MICOS complex",
    "1185": "tail-anchored membrane protein insertion into ER membrane",
    "1186": "K48-linked deubiquitinase activity",
    "1187": "Dsl1/NZR complex",
    "1188": "long-chain fatty acid luciferin component ligase activity",
    "1189": "catechol 1,2-dioxygenase activity",
    "1190": "rRNA methylation",
    "1191": "tRNA guanylyltransferase activity",
    "1192": "response to organic substance",
    "1193": "CDP-glycerol glycerophosphotransferase activity",
    "1194": "protein phosphatase binding",
    "1195": "dolichol-linked oligosaccharide biosynthetic process",
    "1196": "modulation by virus of host cellular process",
    "1197": "viral release via pore formation in host cell membrane",
    "1198": "4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthase activity",
    "1199": "plant-type cell wall organization",
    "1200": "glutaminyl-tRNA aminoacylation",
    "1201": "negative regulation of ATP-dependent activity",
    "1202": "signal peptidase complex",
    "1203": "protein glycosylation",
    "1204": "Actinobacterium-type cell wall biogenesis",
    "1205": "guanosine tetraphosphate metabolic process",
    "1206": "phosphatidylglycerophosphatase activity",
    "1207": "protein secretion by the type IV secretion system",
    "1208": "sexual reproduction",
    "1209": "lipid A biosynthetic process",
    "1210": "peroxisome",
    "1211": "chorion-containing eggshell formation",
    "1212": "mitochondrial proton-transporting ATP synthase complex, catalytic sector F(1)",
    "1213": "regulation of response to reactive oxygen species",
    "1214": "quorum sensing",
    "1215": "secondary metabolic process",
    "1216": "phenol 2-monooxygenase activity",
    "1217": "signaling receptor activity",
    "1218": "polyketide biosynthetic process",
    "1219": "chromatin remodeling",
    "1220": "negative regulation of Wnt signaling pathway",
    "1221": "gonad development",
    "1222": "radial spoke",
    "1223": "respiratory electron transport chain",
    "1224": "glycoprotein catabolic process",
    "1225": "phosphoprotein phosphatase activity",
    "1226": "glycine reductase complex",
    "1227": "beta-1,4-mannosylglycoprotein 4-beta-N-acetylglucosaminyltransferase activity",
    "1228": "chromatin organization",
    "1229": "ecdysis, chitin-based cuticle",
    "1230": "glutathione specific gamma-glutamylcyclotransferase activity",
    "1231": "hydrotropism",
    "1232": "elongator holoenzyme complex",
    "1233": "viral protein processing",
    "1234": "mating type determination",
    "1235": "induction by virus of host apoptotic process",
    "1236": "proton transmembrane transport",
    "1237": "thiol oxidase activity",
    "1238": "sarcoglycan complex",
    "1239": "Ino80 complex",
    "1240": "mitochondrial fusion",
    "1241": "COPII vesicle coat",
    "1242": "tRNA (adenine(22)-N1)-methyltransferase activity",
    "1243": "carbon-sulfur lyase activity",
    "1244": "cellulose microfibril organization",
    "1245": "purine-rich negative regulatory element binding",
    "1246": "calcium-mediated signaling",
    "1247": "four-way junction DNA binding",
    "1248": "nuclear chromosome",
    "1249": "visual perception",
    "1250": "vesicle fusion with Golgi apparatus",
    "1251": "mannose binding",
    "1252": "peroxisome organization",
    "1253": "glutamate synthase activity",
    "1254": "regulation of G protein-coupled receptor signaling pathway",
    "1255": "microtubule-based process",
    "1256": "dolichyl pyrophosphate Glc2Man9GlcNAc2 alpha-1,2-glucosyltransferase activity",
    "1257": "tissue regeneration",
    "1258": "spliceosomal snRNP assembly",
    "1259": "blue light photoreceptor activity",
    "1260": "regulation of viral transcription",
    "1261": "arginine N-succinyltransferase activity",
    "1262": "glutamine metabolic process",
    "1263": "intramolecular oxidoreductase activity, interconverting aldoses and ketoses",
    "1264": "carbohydrate transport",
    "1265": "nicotinamide riboside transport",
    "1266": "anaerobic electron transport chain",
    "1267": "protein phosphatase inhibitor activity",
    "1268": "heme transport",
    "1269": "N-succinylarginine dihydrolase activity",
    "1270": "positive regulation of growth",
    "1271": "alpha-1,4-mannosyltransferase activity",
    "1272": "ubiquinone biosynthetic process",
    "1273": "glutathione gamma-glutamylcysteinyltransferase activity",
    "1274": "monosaccharide binding",
    "1275": "poly(ADP-ribose) glycohydrolase activity",
    "1276": "histone methyltransferase activity",
    "1277": "2-dehydro-3-deoxygalactonokinase activity",
    "1278": "hormone-mediated signaling pathway",
    "1279": "copper chaperone activity",
    "1280": "alpha-1,6-mannosylglycoprotein 2-beta-N-acetylglucosaminyltransferase activity",
    "1281": "mitochondrial respiratory chain complex I assembly",
    "1282": "glutamate catabolic process to 2-oxoglutarate",
    "1283": "gamma-glutamyl carboxylase activity",
    "1284": "eukaryotic translation initiation factor 3 complex",
    "1285": "glutaminyl-peptide cyclotransferase activity",
    "1286": "maintenance of CRISPR repeat elements",
    "1287": "structural constituent of tooth enamel",
    "1288": "peptidyl-amino acid modification",
    "1289": "gas vesicle organization",
    "1290": "RNA polymerase III complex",
    "1291": "phenylacetate catabolic process",
    "1292": "peptide modification",
    "1293": "motile cilium",
    "1294": "photosynthesis, light reaction",
    "1295": "inositol catabolic process",
    "1296": "polysaccharide transport",
    "1297": "proteasome regulatory particle, lid subcomplex",
    "1298": "iron ion transport",
    "1299": "adenylate kinase activity",
    "1300": "urea metabolic process",
    "1301": "potassium ion transmembrane transport",
    "1302": "protein splicing",
    "1303": "regulation of cell division",
    "1304": "protein-L-histidine N-pros-methyltransferase activity",
    "1305": "transcription by RNA polymerase I",
    "1306": "CENP-A containing chromatin assembly",
    "1307": "oligosaccharyltransferase I complex",
    "1308": "intracellular copper ion homeostasis",
    "1309": "regulation of bacterial-type flagellum assembly",
    "1310": "malonyl-CoA decarboxylase activity",
    "1311": "bitter taste receptor activity",
    "1312": "tubulin N-acetyltransferase activity",
    "1313": "transcription initiation at RNA polymerase I promoter",
    "1314": "sodium channel inhibitor activity",
    "1315": "chylomicron",
    "1316": "regulation of phosphorylation",
    "1317": "enzyme regulator activity",
    "1318": "deoxyribonuclease IV (phage-T4-induced) activity",
    "1319": "2-oxoglutarate-dependent dioxygenase activity",
    "1320": "vacuole",
    "1321": "(1->6)-beta-D-glucan biosynthetic process",
    "1322": "mitochondrial respirasome",
    "1323": "bacteriochlorophyll biosynthetic process",
    "1324": "oligosaccharide biosynthetic process",
    "1325": "endoplasmic reticulum",
    "1326": "corticotropin-releasing hormone binding",
    "1327": "oxidoreductase activity, acting on the CH-NH group of donors",
    "1328": "lysosomal membrane",
    "1329": "thyrotropin-releasing hormone activity",
    "1330": "dystrophin-associated glycoprotein complex",
    "1331": "eukaryotic initiation factor 4E binding",
    "1332": "nuclear origin of replication recognition complex",
    "1333": "translation initiation factor binding",
    "1334": "coagulation",
    "1335": "positive regulation of mitochondrial translation",
    "1336": "cellularization",
    "1337": "synaptonemal complex",
    "1338": "lysozyme activity",
    "1339": "dynactin complex",
    "1340": "viral RNA genome packaging",
    "1341": "phospholipase C activity",
    "1342": "microfibril",
    "1343": "microtubule polymerization",
    "1344": "branched-chain amino acid transmembrane transporter activity",
    "1345": "viral occlusion body",
    "1346": "Z disc",
    "1347": "mitotic spindle assembly checkpoint signaling",
    "1348": "response to unfolded protein",
    "1349": "serine-type exopeptidase activity",
    "1350": "THO complex part of transcription export complex",
    "1351": "cytoplasmic microtubule organization",
    "1352": "molybdate ion transmembrane transporter activity",
    "1353": "protein pupylation",
    "1354": "peroxisome fission",
    "1355": "deadenylation-dependent decapping of nuclear-transcribed mRNA",
    "1356": "magnesium ion transport",
    "1357": "microtubule cytoskeleton",
    "1358": "mitochondrion",
    "1359": "Golgi cisterna membrane",
    "1360": "glycogen biosynthetic process",
    "1361": "selenium binding",
    "1362": "synapse",
    "1363": "phosphatidylinositol-4-phosphate binding",
    "1364": "S-adenosylmethionine-dependent methyltransferase activity",
    "1365": "collagen binding",
    "1366": "mating behavior",
    "1367": "mitotic chromosome condensation",
    "1368": "positive regulation of transcription elongation by RNA polymerase II",
    "1369": "plasmid partitioning",
    "1370": "5'-nucleotidase activity",
    "1371": "melanin-concentrating hormone activity",
    "1372": "syntaxin binding",
    "1373": "kinetochore assembly",
    "1374": "anaphase-promoting complex-dependent catabolic process",
    "1375": "organic phosphonate metabolic process",
    "1376": "3-keto-5-aminohexanoate cleavage activity",
    "1377": "DNA protection",
    "1378": "actin filament polymerization",
    "1379": "mitotic cell cycle",
    "1380": "multivesicular body sorting pathway",
    "1381": "pheromone biosynthetic process",
    "1382": "ceramide metabolic process",
    "1383": "2',3'-cyclic-nucleotide 3'-phosphodiesterase activity",
    "1384": "N-terminal protein amino acid methylation",
    "1385": "acyl-CoA dehydrogenase activity",
    "1386": "oxidoreductase activity, acting on NAD(P)H, quinone or similar compound as acceptor",
    "1387": "beta-catenin binding",
    "1388": "protein secretion by the type III secretion system",
    "1389": "aryl sulfotransferase activity",
    "1390": "extracellular organelle",
    "1391": "neuropeptide hormone activity",
    "1392": "telomere maintenance",
    "1393": "ethanolamine ammonia-lyase activity",
    "1394": "peptide transmembrane transporter activity",
    "1395": "viral outer capsid",
    "1396": "regulation of actin filament polymerization",
    "1397": "oxidoreductase activity, acting on the CH-CH group of donors, iron-sulfur protein as acceptor",
    "1398": "preribosome, small subunit precursor",
    "1399": "FtsZ-dependent cytokinesis",
    "1400": "organic phosphonate catabolic process",
    "1401": "regulation of embryonic development",
    "1402": "glycine N-acyltransferase activity",
    "1403": "ribose-5-phosphate isomerase activity",
    "1404": "malate dehydrogenase (quinone) activity",
    "1405": "exocyst",
    "1406": "blood coagulation",
    "1407": "3-hydroxyanthranilate 3,4-dioxygenase activity",
    "1408": "deoxyribonuclease inhibitor activity",
    "1409": "intracellular transport of virus",
    "1410": "nucleoside diphosphate phosphatase activity",
    "1411": "inositol pentakisphosphate 2-kinase activity",
    "1412": "oleate hydratase activity",
    "1413": "positive regulation of catalytic activity",
    "1414": "hemolysis in another organism",
    "1415": "BRCA1-A complex",
    "1416": "L-rhamnose isomerase activity",
    "1417": "taste receptor activity",
    "1418": "positive regulation of cytoskeleton organization",
    "1419": "septin ring assembly",
    "1420": "tRNA-(2-methylthio-N-6-(cis-hydroxy)isopentenyl adenosine)-hydroxylase activity",
    "1421": "anaerobic cobalamin biosynthetic process",
    "1422": "nucleotide metabolic process",
    "1423": "protein-phycocyanobilin linkage",
    "1424": "lymphocyte activation",
    "1425": "melanin biosynthetic process",
    "1426": "riboflavin transport",
    "1427": "generation of catalytic spliceosome for second transesterification step",
    "1428": "translation initiation factor activity",
    "1429": "manganese ion binding",
    "1430": "[isocitrate dehydrogenase (NADP+)] kinase activity",
    "1431": "host outer membrane",
    "1432": "response to light stimulus",
    "1433": "carbon-oxygen lyase activity, acting on phosphates",
    "1434": "establishment of competence for transformation",
    "1435": "ectoine biosynthetic process",
    "1436": "cholesterol metabolic process",
    "1437": "calcium channel inhibitor activity",
    "1438": "cellular component organization",
    "1439": "anaerobic glutamate catabolic process",
    "1440": "pore complex",
    "1441": "actin cytoskeleton organization",
    "1442": "adult feeding behavior",
    "1443": "mitochondrial electron transport, NADH to ubiquinone",
    "1444": "AP-3 adaptor complex",
    "1445": "hexose transmembrane transport",
    "1446": "clathrin light chain binding",
    "1447": "response to parathyroid hormone",
    "1448": "response to acidic pH",
    "1449": "positive regulation of apoptotic process",
    "1450": "angiotensin-activated signaling pathway",
    "1451": "negative regulation of biosynthetic process",
    "1452": "low-density lipoprotein particle receptor binding",
    "1453": "cell population proliferation",
    "1454": "red chlorophyll catabolite reductase activity",
    "1455": "glucose catabolic process",
    "1456": "biological process involved in symbiotic interaction",
    "1457": "pyrimidine nucleotide biosynthetic process",
    "1458": "mitochondrial chromosome",
    "1459": "ATPase-coupled transmembrane transporter activity",
    "1460": "serine O-acetyltransferase activity",
    "1461": "UDP-glucose:glycoprotein glucosyltransferase activity",
    "1462": "amine dehydrogenase activity",
    "1463": "aconitate hydratase activity",
    "1464": "IMP 5'-nucleotidase activity",
    "1465": "response to xenobiotic stimulus",
    "1466": "intracellular iron ion homeostasis",
    "1467": "sodium:proton antiporter activity",
    "1468": "carbohydrate catabolic process",
    "1469": "protein domain specific binding",
    "1470": "ryanodine-sensitive calcium-release channel activity",
    "1471": "molybdopterin synthase complex",
    "1472": "RNA exonuclease activity",
    "1473": "cytochrome bo3 ubiquinol oxidase activity",
    "1474": "female sex differentiation",
    "1475": "phosphorelay response regulator activity",
    "1476": "response to hormone",
    "1477": "voltage-gated sodium channel activity",
    "1478": "bacteriocin transport",
    "1479": "circadian rhythm",
    "1480": "negative regulation of termination of DNA-templated transcription",
    "1481": "translation regulator activity",
    "1482": "2'-deoxyribonucleotide metabolic process",
    "1483": "FMN adenylyltransferase activity",
    "1484": "regulation of exit from mitosis",
    "1485": "L-amino acid efflux transmembrane transporter activity",
    "1486": "regulation of growth",
    "1487": "positive regulation of cell cycle G2/M phase transition",
    "1488": "antigen processing and presentation",
    "1489": "double-stranded DNA 5'-3' exodeoxyribonuclease activity",
    "1490": "protein-containing complex assembly",
    "1491": "phage shock",
    "1492": "hyaluronan metabolic process",
    "1493": "regulation of store-operated calcium entry",
    "1494": "microtubule minus-end binding",
    "1495": "structural constituent of cytoskeleton",
    "1496": "bacteriocin immunity",
    "1497": "viral DNA genome replication",
    "1498": "peptidoglycan turnover",
    "1499": "nucleolus",
    "1500": "microtubule cytoskeleton organization",
    "1501": "regulation of mitochondrial mRNA stability",
    "1502": "modulation of process of another organism",
    "1503": "organic phosphonate transport",
    "1504": "RNA catabolic process",
    "1505": "lipid storage",
    "1506": "carbohydrate transmembrane transporter activity",
    "1507": "mRNA 3'-end processing",
    "1508": "RNA-templated DNA biosynthetic process",
    "1509": "cell differentiation",
    "1510": "transcription corepressor activity",
    "1511": "DNA-(apurinic or apyrimidinic site) endonuclease activity",
    "1512": "ligase activity, forming carbon-nitrogen bonds",
    "1513": "U3 snoRNA binding",
    "1514": "elongin complex",
    "1515": "enterobacterial common antigen biosynthetic process",
    "1516": "negative regulation of apoptotic process",
    "1517": "pectin catabolic process",
    "1518": "glycolipid biosynthetic process",
    "1519": "mRNA 5'-cap (guanine-N7-)-methyltransferase activity",
    "1520": "deoxyribonucleotide catabolic process",
    "1521": "phospholipase A2 activity",
    "1522": "response to arsenic-containing substance",
    "1523": "apoplast",
    "1524": "COPI vesicle coat",
    "1525": "iron ion transmembrane transporter activity",
    "1526": "regulation of pH",
    "1527": "response to redox state",
    "1528": "mitochondrial ribosome",
    "1529": "galactosylceramide sulfotransferase activity",
    "1530": "keratinocyte differentiation",
    "1531": "perturbation by virus of host apoptosis",
    "1532": "hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides",
    "1533": "endoplasmic reticulum organization",
    "1534": "magnesium protoporphyrin IX methyltransferase activity",
    "1535": "chloroplast",
    "1536": "violaxanthin de-epoxidase activity",
    "1537": "type II interferon-mediated signaling pathway",
    "1538": "prenylcysteine catabolic process",
    "1539": "chromosome segregation",
    "1540": "protein import into peroxisome membrane",
    "1541": "poly-hydroxybutyrate biosynthetic process",
    "1542": "extracellular matrix binding",
    "1543": "biomineral tissue development",
    "1544": "secretion",
    "1545": "phosphatidylinositol 3-kinase binding",
    "1546": "cyclic-di-GMP binding",
    "1547": "immune system process",
    "1548": "negative regulation of endopeptidase activity",
    "1549": "phosphate ion transmembrane transport",
    "1550": "extracellular matrix organization",
    "1551": "adhesion of symbiont to host cell",
    "1552": "photosynthesis, dark reaction",
    "1553": "BBSome",
    "1554": "methylamine metabolic process",
    "1555": "dolichol metabolic process",
    "1556": "menaquinone biosynthetic process",
    "1557": "DNA-templated viral transcription",
    "1558": "ATP synthesis coupled electron transport",
    "1559": "cytoplasmic side of plasma membrane",
    "1560": "oxidoreductase activity, acting on X-H and Y-H to form an X-Y bond, with a disulfide as acceptor",
    "1561": "NAD+-dinitrogen-reductase ADP-D-ribosyltransferase activity",
    "1562": "acetylcholine receptor inhibitor activity",
    "1563": "polyketide metabolic process",
    "1564": "negative regulation of bacterial-type flagellum assembly",
    "1565": "negative regulation of cell population proliferation",
    "1566": "viral latency",
    "1567": "cytoskeleton-dependent cytokinesis",
    "1568": "mycotoxin biosynthetic process",
    "1569": "fucosylation",
    "1570": "bone remodeling",
    "1571": "lipid transport",
    "1572": "D-alanine-D-alanine ligase activity",
    "1573": "glycerol-3-phosphate dehydrogenase [NAD(P)+] activity",
    "1574": "activation of GTPase activity",
    "1575": "negative regulation of phosphoprotein phosphatase activity",
    "1576": "trehalose catabolic process",
    "1577": "DNA-templated transcription termination",
    "1578": "Holliday junction helicase complex",
    "1579": "proton-transporting ATP synthase complex assembly",
    "1580": "spore wall",
    "1581": "meiotic chromosome segregation",
    "1582": "monoatomic anion transmembrane transporter activity",
    "1583": "nitrate assimilation",
    "1584": "Smc5-Smc6 complex",
    "1585": "phospholipid binding",
    "1586": "pilus assembly",
    "1587": "Notch signaling pathway",
    "1588": "ferrous iron transmembrane transporter activity",
    "1589": "rhythmic process",
    "1590": "guanylate cyclase activity",
    "1591": "aromatic amino acid family catabolic process",
    "1592": "Rab geranylgeranyltransferase activity",
    "1593": "identical protein binding",
    "1594": "protein complex oligomerization",
    "1595": "glucosidase activity",
    "1596": "mannose metabolic process",
    "1597": "ribosome binding",
    "1598": "centromere complex assembly",
    "1599": "negative regulation of signal transduction",
    "1600": "poly(A)+ mRNA export from nucleus",
    "1601": "alpha-amylase activity",
    "1602": "cyclic-nucleotide phosphodiesterase activity",
    "1603": "IMP cyclohydrolase activity",
    "1604": "pyrimidine nucleoside metabolic process",
    "1605": "oxo-acid-lyase activity",
    "1606": "regulation of fatty acid metabolic process",
    "1607": "cadmium ion binding",
    "1608": "viral release from host cell",
    "1609": "L-fucose isomerase activity",
    "1610": "H4/H2A histone acetyltransferase complex",
    "1611": "endoplasmic reticulum lumen",
    "1612": "RNA biosynthetic process",
    "1613": "protein import into nucleus",
    "1614": "mRNA binding",
    "1615": "retrograde transport, endosome to Golgi",
    "1616": "endocytosis",
    "1617": "oxidized purine nucleobase lesion DNA N-glycosylase activity",
    "1618": "nematocyst",
    "1619": "potassium ion binding",
    "1620": "endoplasmic reticulum calcium ion homeostasis",
    "1621": "protein transmembrane transporter activity",
    "1622": "protein insertion into mitochondrial inner membrane from matrix",
    "1623": "replication fork processing",
    "1624": "cytolysis in another organism",
    "1625": "microtubule organizing center",
    "1626": "inositol biosynthetic process",
    "1627": "host cell endoplasmic reticulum membrane",
    "1628": "defense response to other organism",
    "1629": "protein import into mitochondrial matrix",
    "1630": "pyrroloquinoline quinone biosynthetic process",
    "1631": "cell junction",
    "1632": "branched-chain amino acid biosynthetic process",
    "1633": "pyrimidine nucleobase biosynthetic process",
    "1634": "transcriptional attenuation by ribosome",
    "1635": "leucine biosynthetic process",
    "1636": "protein serine/threonine kinase activity",
    "1637": "nuclear exosome (RNase complex)",
    "1638": "response to toxic substance",
    "1639": "potassium channel inhibitor activity",
    "1640": "innate immune response",
    "1641": "antifungal humoral response",
    "1642": "regulation of catalytic activity",
    "1643": "mitochondrion inheritance",
    "1644": "histone H3K79 methyltransferase activity",
    "1645": "regulation of viral process",
    "1646": "regulation of female receptivity, post-mating",
    "1647": "endospore-forming forespore",
    "1648": "intra-Golgi vesicle-mediated transport",
    "1649": "positive regulation of single-species biofilm formation",
    "1650": "hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amidines",
    "1651": "cytidine deaminase activity",
    "1652": "[citrate (pro-3S)-lyase] ligase activity",
    "1653": "histone lysine methylation",
    "1654": "nodulation",
    "1655": "threonine biosynthetic process",
    "1656": "spindle microtubule",
    "1657": "TIM23 mitochondrial import inner membrane translocase complex",
    "1658": "RNA ligase (ATP) activity",
    "1659": "retrograde vesicle-mediated transport, Golgi to endoplasmic reticulum",
    "1660": "acetyl-CoA carboxylase activity",
    "1661": "N6-(1,2-dicarboxyethyl)AMP AMP-lyase (fumarate-forming) activity",
    "1662": "calmodulin-dependent protein kinase activity",
    "1663": "procollagen-proline 4-dioxygenase activity",
    "1664": "semaphorin receptor activity",
    "1665": "peptide transport",
    "1666": "enoyl-[acyl-carrier-protein] reductase (NADH) activity",
    "1667": "chlorophyll biosynthetic process",
    "1668": "regulation of protein catabolic process",
    "1669": "sensory perception of taste",
    "1670": "plasma membrane organization",
    "1671": "biological process involved in interspecies interaction between organisms",
    "1672": "chitin synthase activity",
    "1673": "single-stranded DNA 3'-5' DNA exonuclease activity",
    "1674": "oxidoreductase activity, acting on NAD(P)H, oxygen as acceptor",
    "1675": "chlorophyllide a oxygenase [overall] activity",
    "1676": "lipopolysaccharide 3-alpha-galactosyltransferase activity",
    "1677": "RNA-protein covalent cross-linking",
    "1678": "excinuclease ABC activity",
    "1679": "rRNA (guanine-N2-)-methyltransferase activity",
    "1680": "sucrose-phosphate phosphatase activity",
    "1681": "UDP-glucose 4-epimerase activity",
    "1682": "squalene monooxygenase activity",
    "1683": "aflatoxin biosynthetic process",
    "1684": "steroid biosynthetic process",
    "1685": "shikimate 3-dehydrogenase (NADP+) activity",
    "1686": "2-isopropylmalate synthase activity",
    "1687": "tetrahydrodipicolinate N-acetyltransferase activity",
    "1688": "DNA replication factor C complex",
    "1689": "regulation of termination of DNA-templated transcription",
    "1690": "dipeptidyl-peptidase activity",
    "1691": "beta-galactosidase activity",
    "1692": "galactose metabolic process",
    "1693": "hydroxymethylglutaryl-CoA synthase activity",
    "1694": "3-oxoacyl-[acyl-carrier-protein] synthase activity",
    "1695": "inositol phosphoceramide metabolic process",
    "1696": "protein homodimerization activity",
    "1697": "nuclear protein quality control by the ubiquitin-proteasome system",
    "1698": "PAM complex, Tim23 associated import motor",
    "1699": "RSC-type complex",
    "1700": "mitochondrion autophagosome adaptor activity",
    "1701": "mitochondrial inner membrane",
    "1702": "regulation of cyclin-dependent protein serine/threonine kinase activity",
    "1703": "ribonuclease H2 complex",
    "1704": "meiotic cell cycle",
    "1705": "mitochondrial RNA metabolic process",
    "1706": "vacuolar proton-transporting V-type ATPase complex assembly",
    "1707": "maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)",
    "1708": "mitotic sister chromatid segregation",
    "1709": "histone deacetylation",
    "1710": "mitotic spindle",
    "1711": "attachment of spindle microtubules to kinetochore",
    "1712": "poly(A)-specific ribonuclease activity",
    "1713": "UFM1 conjugating enzyme activity",
    "1714": "platelet activation",
    "1715": "carbon-nitrogen lyase activity",
    "1716": "omega peptidase activity",
    "1717": "intermembrane lipid transfer",
    "1718": "cellular response to iron ion starvation",
    "1719": "nuclear export signal receptor activity",
    "1720": "regulation of sporulation resulting in formation of a cellular spore",
    "1721": "protein-containing complex binding",
    "1722": "SMAD binding",
    "1723": "nuclear receptor binding",
    "1724": "NEDD8 activating enzyme activity",
    "1725": "MHC class II protein binding",
    "1726": "nuclear pore",
    "1727": "threonine-tRNA ligase activity",
    "1728": "transforming growth factor beta receptor activity, type II",
    "1729": "peptidyl-histidine phosphorylation",
    "1730": "non-membrane spanning protein tyrosine kinase activity",
    "1731": "regulation of TOR signaling",
    "1732": "icosahedral viral capsid",
    "1733": "ubiquitin protein ligase activity",
    "1734": "oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, with 2-oxoglutarate as one donor, and the other dehydrogenated",
    "1735": "cell-cell adhesion",
    "1736": "negative regulation of cell killing",
    "1737": "NAD+-diphthamide ADP-ribosyltransferase activity",
    "1738": "protein-glutamine gamma-glutamyltransferase activity",
    "1739": "negative regulation of phagocytosis",
    "1740": "type III protein secretion system complex",
    "1741": "porphyrin-containing compound metabolic process",
    "1742": "histone acetyltransferase complex",
    "1743": "positive regulation of canonical NF-kappaB signal transduction",
    "1744": "P-type potassium:proton transporter activity",
    "1745": "activation of cysteine-type endopeptidase activity involved in apoptotic process",
    "1746": "transmembrane signaling receptor activity",
    "1747": "condensed nuclear chromosome",
    "1748": "virus tail, fiber",
    "1749": "virus tail, baseplate",
    "1750": "double-strand break repair via homologous recombination",
    "1751": "selenocysteine incorporation",
    "1752": "nucleosome binding",
    "1753": "negative regulation of growth",
    "1754": "tRNA thio-modification",
    "1755": "phosphatidate cytidylyltransferase activity",
    "1756": "ruffle",
    "1757": "light absorption",
    "1758": "mRNA export from nucleus",
    "1759": "heme catabolic process",
    "1760": "negative regulation of transcription by RNA polymerase III",
    "1761": "Golgi vesicle transport",
    "1762": "prolyl-tRNA aminoacylation",
    "1763": "protein phosphatase 4 complex",
    "1764": "regulation of DNA methylation",
    "1765": "cysteine-tRNA ligase activity",
    "1766": "evasion by virus of host restriction-modification system",
    "1767": "arabinan metabolic process",
    "1768": "cell death",
    "1769": "chemokine binding",
    "1770": "adhesion of symbiont to host",
    "1771": "apoptotic DNA fragmentation",
    "1772": "erythrocyte differentiation",
    "1773": "production of molecular mediator involved in inflammatory response",
    "1774": "regulation of immune system process",
    "1775": "sialic acid binding",
    "1776": "cytokinin metabolic process",
    "1777": "RNA polymerase II cis-regulatory region sequence-specific DNA binding",
    "1778": "proteasome-mediated ubiquitin-dependent protein catabolic process",
    "1779": "nucleoside binding",
    "1780": "carbon-oxygen lyase activity, acting on polysaccharides",
    "1781": "peptide biosynthetic process",
    "1782": "acetaldehyde dehydrogenase (acetylating) activity",
    "1783": "aromatic amino acid metabolic process",
    "1784": "fatty acid beta-oxidation using acyl-CoA dehydrogenase",
    "1785": "phytochelatin biosynthetic process",
    "1786": "intracellular membrane-bounded organelle",
    "1787": "3'-5' DNA helicase activity",
    "1788": "regulation of sporulation",
    "1789": "microtubule anchoring",
    "1790": "nuclear-transcribed mRNA catabolic process, nonsense-mediated decay",
    "1791": "SREBP signaling pathway",
    "1792": "nucleoplasmic THO complex",
    "1793": "RNA methylation",
    "1794": "kinetochore",
    "1795": "mannosylglycerate biosynthetic process",
    "1796": "Slx1-Slx4 complex",
    "1797": "glucosylglycerol 3-phosphatase activity",
    "1798": "delta DNA polymerase complex",
    "1799": "SWI/SNF complex",
    "1800": "thiamine transmembrane transporter activity",
    "1801": "nucleic acid phosphodiester bond hydrolysis",
    "1802": "nuclear speck",
    "1803": "regulation of ATP-dependent activity",
    "1804": "hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amides",
    "1805": "ARC complex",
    "1806": "Ctf18 RFC-like complex",
    "1807": "neuronal signal transduction",
    "1808": "cytoskeletal anchor activity",
    "1809": "UFM1 ligase activity",
    "1810": "regulation of alternative mRNA splicing, via spliceosome",
    "1811": "U6 snRNA 3'-end processing",
    "1812": "exocytosis",
    "1813": "cristae formation",
    "1814": "positive regulation of protein dephosphorylation",
    "1815": "MKS complex",
    "1816": "mitochondrial intermembrane space",
    "1817": "mitochondrial large ribosomal subunit",
    "1818": "Golgi organization",
    "1819": "phosphatidylinositol-4,5-bisphosphate 4-phosphatase activity",
    "1820": "phagophore assembly site",
    "1821": "post-translational protein targeting to membrane, translocation",
    "1822": "mitochondrial cytochrome c oxidase assembly",
    "1823": "single-stranded DNA 5'-3' DNA exonuclease activity",
    "1824": "RZZ complex",
    "1825": "ATP adenylyltransferase activity",
    "1826": "lactate racemase activity",
    "1827": "phosphatidate phosphatase activity",
    "1828": "dioxygenase activity",
    "1829": "protein targeting to ER",
    "1830": "Atg1/ULK1 kinase complex",
    "1831": "DRM complex",
    "1832": "protein-arginine rhamnosyltransferase activity",
    "1833": "nuclear membrane",
    "1834": "thiamine-phosphate diphosphorylase activity",
    "1835": "NAD glycohydrolase activity",
    "1836": "2,3-bisphosphoglycerate-independent phosphoglycerate mutase activity",
    "1837": "WASH complex",
    "1838": "lysosome localization",
    "1839": "calcium import into the mitochondrion",
    "1840": "SAGA complex",
    "1841": "positive regulation of plasminogen activation",
    "1842": "TIM22 mitochondrial import inner membrane insertion complex",
    "1843": "proteasome assembly",
    "1844": "response to pheromone",
    "1845": "positive regulation of SREBP signaling pathway",
    "1846": "regulation of protein ADP-ribosylation",
    "1847": "cobalamin metabolic process",
    "1848": "lipid droplet",
    "1849": "intrinsic apoptotic signaling pathway",
    "1850": "ESCRT I complex",
    "1851": "acyl-coenzyme A diphosphatase activity",
    "1852": "RNA 7-methylguanosine cap binding",
    "1853": "G protein-coupled peptide receptor activity",
    "1854": "telomere maintenance via telomerase",
    "1855": "mitotic sister chromatid cohesion",
    "1856": "rRNA (uridine-N3-)-methyltransferase activity",
    "1857": "regulation of ribosomal protein gene transcription by RNA polymerase II",
    "1858": "histone binding",
    "1859": "transcription elongation factor complex",
    "1860": "DNA polymerase activity",
    "1861": "ubiquitin-ubiquitin ligase activity",
    "1862": "porphyrin-containing compound biosynthetic process",
    "1863": "tricarboxylic acid cycle",
    "1864": "transmembrane receptor protein tyrosine kinase signaling pathway",
    "1865": "mannose-6-phosphate isomerase activity",
    "1866": "exosome (RNase complex)",
    "1867": "TORC1 complex",
    "1868": "valine-tRNA ligase activity",
    "1869": "serine-type aminopeptidase activity",
    "1870": "DNA damage response",
    "1871": "nuclear inner membrane",
    "1872": "phosphatidylinositol 3-kinase complex, class IB",
    "1873": "signal peptide processing",
    "1874": "TTT Hsp90 cochaperone complex",
    "1875": "host cell postsynaptic membrane",
    "1876": "SAM complex",
    "1877": "negative regulation of actin nucleation",
    "1878": "4 iron, 4 sulfur cluster binding",
    "1879": "acetylcholine receptor binding",
    "1880": "proteasome core complex, alpha-subunit complex",
    "1881": "U6 snRNA binding",
    "1882": "U5 snRNA binding",
    "1883": "butyrate metabolic process",
    "1884": "L-ascorbic acid binding",
    "1885": "biogenic amine metabolic process",
    "1886": "'de novo' protein folding",
    "1887": "regulation of proteasomal protein catabolic process",
    "1888": "xenobiotic detoxification by transmembrane export across the plasma membrane",
    "1889": "hydrolase activity, acting on acid halide bonds, in C-halide compounds",
    "1890": "cellular response to acidic pH",
    "1891": "energy reserve metabolic process",
    "1892": "90S preribosome",
    "1893": "respirasome",
    "1894": "mitochondrial oxoglutarate dehydrogenase complex",
    "1895": "RNA polymerase II general transcription initiation factor activity",
    "1896": "regulation of cilium movement",
    "1897": "viral inner membrane",
    "1898": "mitochondrial inner membrane peptidase complex",
    "1899": "SMN-Sm protein complex",
    "1900": "orexigenic neuropeptide QRFP receptor binding",
    "1901": "alginic acid biosynthetic process",
    "1902": "photosystem II assembly",
    "1903": "aldehyde decarbonylase activity",
    "1904": "mediator complex",
    "1905": "virus-mediated pore formation in host cell membrane",
    "1906": "NADH dehydrogenase complex (plastoquinone) assembly",
    "1907": "positive regulation of cyclin-dependent protein serine/threonine kinase activity",
    "1908": "regulation of autophagy",
    "1909": "copper ion homeostasis",
    "1910": "detoxification of cadmium ion",
    "1911": "ATP biosynthetic process",
    "1912": "HDA1 complex",
    "1913": "defense response to Gram-negative bacterium",
    "1914": "termination of RNA polymerase II transcription",
    "1915": "cytoplasmic dynein complex",
    "1916": "flap-structured DNA binding",
    "1917": "phosphatidylinositol-3-phosphate binding",
    "1918": "xylan endo-1,3-beta-xylosidase activity",
    "1919": "kinase binding",
    "1920": "mitochondrial fission",
    "1921": "histone H3K27 methyltransferase activity",
    "1922": "photosynthetic electron transport chain",
    "1923": "G-quadruplex RNA binding",
    "1924": "transcription by RNA polymerase III",
    "1925": "malate transport",
    "1926": "damaged DNA binding",
    "1927": "UDP-glucuronate decarboxylase activity",
    "1928": "enterochelin esterase activity",
    "1929": "beta-fructofuranosidase activity",
    "1930": "potassium channel activity",
    "1931": "oxidoreductase activity, acting on paired donors, with oxidation of a pair of donors resulting in the reduction of molecular oxygen to two molecules of water",
    "1932": "magnesium chelatase activity",
    "1933": "alkaloid metabolic process",
    "1934": "RNA-directed DNA polymerase activity",
    "1935": "[heparan sulfate]-glucosamine N-sulfotransferase activity",
    "1936": "tRNA (guanine(18)-2'-O)-methyltransferase activity",
    "1937": "cytokine binding",
    "1938": "host cell endoplasmic reticulum",
    "1939": "catechol oxidase activity",
    "1940": "IkappaB kinase activity",
    "1941": "negative regulation of peptidase activity",
    "1942": "complement binding",
    "1943": "endo-alpha-(2,8)-sialidase activity",
    "1944": "cell wall macromolecule biosynthetic process",
    "1945": "cerebral cortex development",
    "1946": "structural constituent of carboxysome shell",
    "1947": "smoothened signaling pathway",
    "1948": "intraciliary transport",
    "1949": "cellular response to oxidative stress",
    "1950": "nucleolar ribonuclease P complex",
    "1951": "regulation of cytokinesis",
    "1952": "P-type calcium transporter activity",
    "1953": "Cul4A-RING E3 ubiquitin ligase complex",
    "1954": "organelle organization",
    "1955": "phosphate ion transport",
    "1956": "ERMES complex",
    "1957": "tyrosine 3-monooxygenase activity",
    "1958": "gamma-tubulin complex localization",
    "1959": "T cell receptor signaling pathway",
    "1960": "rRNA (guanine) methyltransferase activity",
    "1961": "MCM complex",
    "1962": "myosin binding",
    "1963": "extrinsic component of intraperoxisomal membrane",
    "1964": "telomere capping",
    "1965": "TRAPPII protein complex",
    "1966": "SAGA-type complex",
    "1967": "cell cortex",
    "1968": "mitochondrial genome maintenance",
    "1969": "cullin family protein binding",
    "1970": "protein homotetramerization",
    "1971": "L-aspartate:2-oxoglutarate aminotransferase activity",
    "1972": "glycopeptide alpha-N-acetylgalactosaminidase activity",
    "1973": "calcium ion import across plasma membrane",
    "1974": "V(D)J recombination",
    "1975": "acetyl-CoA transmembrane transporter activity",
    "1976": "molecular adaptor activity",
    "1977": "regulation of miRNA-mediated gene silencing",
    "1978": "Mis6-Sim4 complex",
    "1979": "1-deoxy-D-xylulose-5-phosphate synthase activity",
    "1980": "photosystem II repair",
    "1981": "beta-lactamase activity",
    "1982": "bacterial-type RNA polymerase core enzyme binding",
    "1983": "ribonucleoside-triphosphate reductase activity",
    "1984": "mitochondrial small ribosomal subunit",
    "1985": "ribonucleoprotein complex",
    "1986": "nitrite reductase [NAD(P)H] activity",
    "1987": "propionyl-CoA carboxylase activity",
    "1988": "cell motility",
    "1989": "olfactory receptor activity",
    "1990": "mRNA polyadenylation",
    "1991": "phosphatase binding",
    "1992": "mRNA stabilization",
    "1993": "dormancy process",
    "1994": "polysaccharide binding",
    "1995": "meiotic DNA double-strand break formation",
    "1996": "cell adhesion involved in single-species biofilm formation",
    "1997": "positive regulation of sporulation resulting in formation of a cellular spore",
    "1998": "capsule polysaccharide biosynthetic process",
    "1999": "mitochondrial membrane",
    "2000": "tocopherol cyclase activity",
    "2001": "respiratory chain complex II",
    "2002": "tRNA-specific adenosine deaminase activity",
    "2003": "preQ1 synthase activity",
    "2004": "ribose phosphate diphosphokinase activity",
    "2005": "oligogalacturonide lyase activity",
    "2006": "epidermis development",
    "2007": "sphingomyelin phosphodiesterase D activity",
    "2008": "HAUS complex",
    "2009": "protein-N-terminal asparagine amidohydrolase activity",
    "2010": "integrator complex",
    "2011": "retinol transmembrane transporter activity",
    "2012": "AP-5 adaptor complex",
    "2013": "mitochondrial respiratory chain complex I",
    "2014": "cornified envelope",
    "2015": "regulation of angiogenesis",
    "2016": "glycerol-3-phosphate O-acyltransferase activity",
    "2017": "1-pyrroline-5-carboxylate dehydrogenase activity",
    "2018": "host cell cytoplasmic vesicle",
    "2019": "defense response to Gram-positive bacterium",
    "2020": "cellular respiration",
    "2021": "axonemal microtubule",
    "2022": "negative regulation of phosphatase activity",
    "2023": "RNA polymerase I transcription regulator complex",
    "2024": "BLOC-1 complex",
    "2025": "mitochondrial proton-transporting ATP synthase complex",
    "2026": "mitochondrion organization",
    "2027": "mitochondrial respiratory chain complex III assembly",
    "2028": "melanosome",
    "2029": "AP-type membrane coat adaptor complex",
    "2030": "microtubule organizing center organization",
    "2031": "activation of innate immune response",
    "2032": "lysosome",
    "2033": "Golgi apparatus",
    "2034": "CatSper complex",
    "2035": "regulation of double-strand break repair via nonhomologous end joining",
    "2036": "alpha-1,6-mannosylglycoprotein 6-beta-N-acetylglucosaminyltransferase activity",
    "2037": "positive regulation of protein phosphorylation",
    "2038": "bile acid and bile salt transport",
    "2039": "tetraspanin-enriched microdomain",
    "2040": "vesicle tethering to Golgi",
    "2041": "nervous system development",
    "2042": "sperm principal piece",
    "2043": "actin filament bundle organization",
    "2044": "recombinational repair",
    "2045": "voltage-gated monoatomic ion channel activity",
    "2046": "protein kinase A regulatory subunit binding",
    "2047": "receptor signaling protein tyrosine kinase activator activity",
    "2048": "regulation of toll-like receptor signaling pathway",
    "2049": "secretory granule membrane",
    "2050": "Fanconi anaemia nuclear complex",
    "2051": "release of cytochrome c from mitochondria",
    "2052": "synaptonemal complex assembly",
    "2053": "positive regulation of immune response",
    "2054": "G protein-coupled receptor binding",
    "2055": "sensory perception of sound",
    "2056": "mitochondrial outer membrane translocase complex",
    "2057": "negative regulation of D-amino-acid oxidase activity",
    "2058": "photoreceptor outer segment membrane",
    "2059": "fat cell differentiation",
    "2060": "sperm plasma membrane",
    "2061": "receptor recycling",
    "2062": "positive regulation of cytokine production",
    "2063": "immune response",
    "2064": "synaptonemal complex organization",
    "2065": "regulation of mitotic cell cycle spindle assembly checkpoint",
    "2066": "connexin binding",
    "2067": "killing of cells of another organism",
    "2068": "intraciliary transport particle A",
    "2069": "transcription repressor complex",
    "2070": "acrosomal vesicle",
    "2071": "spermatogenesis",
    "2072": "snRNA binding",
    "2073": "cellular response to ionizing radiation",
    "2074": "RNA 5'-cap (guanine-N7)-methylation",
    "2075": "tRNA-splicing ligase complex",
    "2076": "negative regulation of double-strand break repair via nonhomologous end joining",
    "2077": "transcription elongation factor activity",
    "2078": "B cell receptor signaling pathway",
    "2079": "RNA polymerase transcription factor SL1 complex",
    "2080": "acrosomal vesicle exocytosis",
    "2081": "histone H4-R3 methylation",
    "2082": "negative regulation of T cell activation",
    "2083": "SMN complex",
    "2084": "cell-cell junction",
    "2085": "non-motile cilium assembly",
    "2086": "apelin receptor binding",
    "2087": "box C/D snoRNP assembly",
    "2088": "EKC/KEOPS complex",
    "2089": "calcium-dependent protein binding",
    "2090": "late endosome membrane",
    "2091": "spliceosomal complex disassembly",
    "2092": "histone acetylation",
    "2093": "chloride transport",
    "2094": "protein ubiquitination",
    "2095": "CST complex",
    "2096": "transcription initiation at RNA polymerase III promoter",
    "2097": "SUMO binding",
    "2098": "negative regulation of centriole replication",
    "2099": "methylated histone binding",
    "2100": "axon guidance",
    "2101": "keratinocyte development",
    "2102": "mitotic cell cycle checkpoint signaling",
    "2103": "FANCM-MHF complex",
    "2104": "regulation of B cell receptor signaling pathway",
    "2105": "circadian regulation of gene expression",
    "2106": "cerebellum development",
    "2107": "mitotic spindle assembly",
    "2108": "regulation of lymphocyte activation",
    "2109": "chondrocyte differentiation",
    "2110": "positive regulation of angiogenesis",
    "2111": "regulation of systemic acquired resistance",
    "2112": "B cell activation",
    "2113": "pollen development",
    "2114": "double-strand break repair involved in meiotic recombination",
    "2115": "centriole replication",
    "2116": "single-species biofilm formation",
    "2117": "negative regulation of cell migration",
    "2118": "nuclear retinoic acid receptor binding",
    "2119": "negative regulation of substrate adhesion-dependent cell spreading",
    "2120": "acrosome reaction",
    "2121": "cartilage development",
    "2122": "positive regulation of enamel mineralization",
    "2123": "dynein complex binding",
    "2124": "neurotrophin TRK receptor signaling pathway",
    "2125": "negative regulation of circadian rhythm",
    "2126": "alpha-tubulin binding",
    "2127": "melanosome transport",
    "2128": "positive regulation of release of cytochrome c from mitochondria",
    "2129": "structural constituent of skin epidermis",
    "2130": "negative regulation of glucose import",
    "2131": "SOSS complex",
    "2132": "regulation of cell growth",
    "2133": "sensory organ development",
    "2134": "centrosome cycle",
    "2135": "dolichyl-phosphate-mannose-protein mannosyltransferase activity",
    "2136": "embryonic development via the syncytial blastoderm",
    "2137": "mitotic cytokinesis",
    "2138": "type IV pilus assembly",
    "2139": "protein trimerization",
    "2140": "type II protein secretion system complex",
    "2141": "positive regulation of myeloid cell differentiation",
    "2142": "acid phosphatase activity",
    "2143": "actin filament organization",
    "2144": "negative regulation of cysteine-type endopeptidase activity involved in apoptotic process",
    "2145": "single-stranded telomeric DNA binding",
    "2146": "Ragulator complex",
    "2147": "Shu complex",
    "2148": "dimethylpropiothetin dethiomethylase activity",
    "2149": "import into cell",
    "2150": "intracellular calcium ion homeostasis",
    "2151": "nitrate transmembrane transport",
    "2152": "molybdate ion transport",
    "2153": "regulation of translation",
    "2154": "mitochondrial crista junction",
    "2155": "mRNA cap binding complex",
    "2156": "fungal-type cell wall organization",
    "2157": "protein localization to spindle pole body",
    "2158": "sleep",
    "2159": "nuclear export",
    "2160": "endoplasmic reticulum inheritance",
    "2161": "sperm individualization",
    "2162": "central plaque of spindle pole body",
    "2163": "regulation of osteoblast differentiation",
    "2164": "neuron projection development",
    "2165": "actin cortical patch",
    "2166": "chromosome, centromeric region",
    "2167": "5S rRNA binding",
    "2168": "acetylcholine-gated channel complex",
    "2169": "regulatory ncRNA-mediated post-transcriptional gene silencing",
    "2170": "outer mitochondrial membrane organization",
    "2171": "positive regulation of heart contraction",
    "2172": "pheromone-dependent signal transduction involved in conjugation with cellular fusion",
    "2173": "vacuole inheritance",
    "2174": "N-acyl homoserine lactone synthase activity",
    "2175": "siderophore transport",
    "2176": "cell adhesion molecule binding",
    "2177": "suppression by virus of host transcription",
    "2178": "histidine metabolic process",
    "2179": "protein secretion by the type VI secretion system",
    "2180": "toxin transport",
    "2181": "phosphatidylinositol 3-kinase complex, class III, type II",
    "2182": "cilium assembly",
    "2183": "mitophagy",
    "2184": "response to reactive oxygen species",
    "2185": "negative regulation of helicase activity",
    "2186": "resolution of meiotic recombination intermediates",
    "2187": "(3R)-hydroxypalmitoyl-[acyl-carrier-protein] dehydratase activity",
    "2188": "suspensor development",
    "2189": "COP9 signalosome assembly",
    "2190": "mitochondrial translation",
    "2191": "intramolecular lyase activity",
    "2192": "Hsp90 protein binding",
    "2193": "CRISPR-cas system",
    "2194": "MAPK cascade",
    "2195": "nuclear cap binding complex",
    "2196": "acetate-CoA ligase (ADP-forming) activity",
    "2197": "cis-Golgi network",
    "2198": "cytoskeleton-dependent intracellular transport",
    "2199": "negative regulation of mitotic cell cycle",
    "2200": "mitochondrial outer membrane translocase complex assembly",
    "2201": "regulation of reciprocal meiotic recombination",
    "2202": "AMP deaminase activity",
    "2203": "C-C chemokine binding",
    "2204": "protein tyrosine phosphatase activity",
    "2205": "SCF-dependent proteasomal ubiquitin-dependent protein catabolic process",
    "2206": "type III site-specific deoxyribonuclease activity",
    "2207": "telomerase catalytic core complex assembly",
    "2208": "regulation of mitotic cytokinesis",
    "2209": "site of double-strand break",
    "2210": "thylakoid membrane organization",
    "2211": "pre-miRNA processing",
    "2212": "rRNA primary transcript binding",
    "2213": "snRNA processing",
    "2214": "RNA cap binding",
    "2215": "deoxyribonuclease I activity",
    "2216": "potassium channel regulator activity",
    "2217": "microtubule",
    "2218": "xylan biosynthetic process",
    "2219": "UNCONDITIONAL"
}